[Content] Coalition: Heliarch Intro (#6647)
endless-sky/endless-sky: data/coalition/heliarch intro.txt

			`	"Much more detailed, these scans are. Certain, I am, that pleased with the results, Consul Aulori will be," the Kimek tells you. "Assisted us greatly you have, Captain."`
			`	They pass along a message from the consul, who thanks you for all your efforts, and asks that you seek out his agents on <planet>, should you acquire any significant information on the Quarg ships or ringworlds. The Heliarchs say their goodbyes, and head on their way.`

mission "Keep Zug Wardragon"
	landing
	invisible
	source "Ring of Wisdom"
	to offer
		or
			has "Heliarch Recon 3-B: done"
			has "Heliarch Recon 3-C: done"
	npc kill
		government "Quarg"
		personality heroic staying uninterested
		system "Zubeneschamali"
		ship "Quarg Wardragon" "Gloref Esa Kurayi"

mission "Heliarch Expedition 1"
	name "Extraterritorial Reconnaissance"
	description "Head through the wormhole in Deneb, land on the planet there, and enter the system with the broken ringworld to get data for the Heliarch."
	landing
	source "Ring of Wisdom"
	stopover "Ruin"
	waypoint "World's End"
	to offer
		has "Ruin: Landing: offered"
		or
			has "Heliarch Recon 3-B: done"
			has "Heliarch Recon 3-C: done"
	on offer
		conversation

> 			`Landing on the <origin> ou remember that the Heliarchs have asked you to share with them any information you get on Quarg ships and ringworlds. Would you like to tell them about the destroyed ring you saw on the other side of the wormhole in Deneb?`

Reviewer: ```suggestion
			`Landing on the <origin> you remember that the Heliarchs have asked you to share with them any information you get on Quarg ships and ringworlds. Would you like to tell them about the destroyed ring you saw on the other side of the wormhole in Deneb?`
```

			`	"Forgive me for the delay you must, colleagues," he says to the other Heliarchs. "Caught up in my evening prayers, I was." He examines the room for a while, briefly glancing at each of the Heliarchs, until his eyes settle on yours. "Told, I was, that some valuable information for us, you have, Captain <last>."`
			choice
				`	"I know of a system where a broken Quarg ringworld is. I figured I should tell you."`
				`	"Would I be paid if I told you about a place where you can find a broken Quarg ringworld?"`
			`	He squints his eyes, then speaks, "Repeat that, could you?" You say it again, and he once more looks puzzled. He gestures to a Heliarch beside him, then takes off the translation box from around his neck, swapping it for theirs.`
			`	He asks again, and you repeat it again. "An... incomplete ringworld, is it?"`
			choice
				`	"I didn't say incomplete, I said broken. I know the difference."`
				`	"Is this some joke? If you don't care about the broken ringworld, just tell me. Don't waste my time."`

> 			`	He leans forward lightly, staring at you. Looking around the table, the other Heliarchs are engaged in a what appears to be a quiet, heated discussion, gesticulating and occasionaly tilting the table with their body weight amidst whispers.`

Reviewer 2: ```suggestion
			`	He leans forward lightly, staring at you. Looking around the table, the other Heliarchs are engaged in a what appears to be a quiet, heated discussion, gesticulating and occasionally tilting the table with their body weight amidst whispers.`
```
Let's get those typos out of the way first.

				accept
	on complete
		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology, and a translation device, to freely converse with the Coalition species."
		set "language: Coalition"
		event "first heliarch unlock"
		set "license: Heliarch"
		conversation
			`When you land, you're met with a bit of ceremony: a trio of Heliarch consuls welcomes you, each carrying a gemstone, an emerald for the Saryd, a ruby for the Kimek, and a sapphire for the Arach. They each pass the precious stones to you, first the Saryd, then the Kimek and lastly the Arach. Cameras and photographers surround the scene, kept at bay by lines of Heliarch guards. It seems your joining the ranks has turned into something of an event.`
			`	When the consuls are done, they have you follow them into the restricted sections of the ring, bringing you to a very large, circular room, somewhat like a colosseum or an auditorium, with hundreds of Heliarch consuls seated. There are also many more camera workers and photographers, though all of them are Heliarch members this time. You're instructed to head to the center.`

> 			`	The consuls who brought yo here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much, this outsider has, shown devotion to our Coalition, this outsider has."`

Reviewer: ```suggestion
			`	The consuls who brought you here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much, this outsider has, shown devotion to our Coalition, this outsider has."`
```

			has "Heliarch Investigation 2: aborted"
			has "Heliarch Investigation 2 :failed"
	on complete
		conversation
			`When you land, a couple of other civilian ships land nearby. Some Kimek leave them and wait outside your own. Kirrie is the only one to leave, and after speaking a bit with the other Kimek, they leave for the city, splitting in two groups.`
			`	"Not fond of outsiders, the people of this world are," Tugroob explains. "Decided to employ undercover Kimek agents, the consul has."`
			`	You wait around for a couple hours as Iplora and Tugroob tinker with your ship's communication devices, keeping in contact with the two groups. They don't mention if either group finds anything, and as night falls, Kirrie returns to the ship on his own.`

mission "Heliarch Recon 1"
	name "Heliarch Reconnaissance"
	description "Fly to all the Quarg planets and stations the Heliarch asked, land on them, letting your modified ship sensors scan their surroundings, then head back to the <destination>."
	minor
	source
		government "Heliarch"
	stopover "Lagrange"
	stopover "Forpelog"
	stopover "Grakhord"
	stopover "Alta Hai"
	stopover "Kuwaru Efreti"
	destination "Ring of Wisdom"

> 	to offer

Reviewer 3: I got the mission even though I am at war with the Quarg, meaning there is no way to land on the planets.
Add a check that Quarg reputation is >= 0 and maybe fail that mission should that no longer be the case.

			label keepdig
			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, and a faint metallic sound echoes. She takes her hands off of you and lays down, now helping you dig out whatever is there. She stands up once the crate is visible. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`
			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast, coming from the the top of the hills, hits the ground near the hole, missing you by just about a meter.`
			`	She yells for you to get down, pulling out her weapon and firing in the direction the shot came from.`

> 			choice

Reviewer 3: small thing: I noticed you give the evidence in both cases for the choice (taking cover and helping) so maybe put it here, before the `choice`?

			choice
				`	"I'm not sure I understand what you're getting at."`
				`	"Obviously I have my limits as for what I would do."`
			`	"A jump drive, you possess. Far away, you can travel. Bother you, the harsh cages of the Quarg do not," the Saryd explains as all three walk toward your ship.`
			`	"Few in number, our own jump drives are, and too risky would it be, for one of our own to venture far out," the Kimek adds.`
			`	"If to help us gain knowledge of the opressors, you wish, lend the Coalition your ship's sensors for a short time, would you?" Asks the Arach.`
			choice
				`	"Are you asking me to spy on the Quarg?"`
				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
					decline
			`	"As espionage, a 'malfunction' qualifies not," the Saryd responds.`
			`	"Merely alter your own ship's sensors, we would. For a time, specially tuned to scan Quarg worlds, they would be," the Kimek clarifies. "No evidence against you, they would have."`
			choice
				`	"Alright, which worlds do you need me to land on?"`
				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
					decline

> 			`	They ask you to show them your map of the galaxy, so you bring them inside your ship, pointing to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`

Author: The player character presumably knows about the Quarg from videos and such media they consumed prior to their first encounter with one, I imagine it'd also be something they know about that the Quarg (in human space) are near Tarazed.
Also don't think it'd be very likely for someone that has the Wanderers mission done wouldn't have at least gone close to the Quarg pocket there (actually think that's straight up impossible given you need FW completed to get to that portion of the Wanderer campaign, and in FW you have to bring Alondo to Tarazed).
In any case I don't think it's necessary to add more conditions for this to trigger, it's reasonable to assume that the player could at least point out the area the Quarg in human space are, I'd say.

> 			`	They ask you to show them your map of the galaxy, so you bring them inside your ship, pointing to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`

Reviewer 3: I don't mind it, what I mostly meant was you don't have to put that Quarg missions as requirement, since the Heliarchs are enemies of the Quarg, why would you need to do missions with them, you could just have "visited planet" for the two ringworlds.

> 			`	They ask you to show them your map of the galaxy, so you bring them inside your ship, pointing to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`

Author: Oh, I just thought it was easier to have that mission as the requirement, but yeah it's a bit restrictive to have this chain locked behind that far into Wanderers.
I'll look into changing the conditions to checking for visits on Alta Hai and Kuwaru Efreti.

					goto lie
				`	"Scanning? I'm sorry, I think my ship just had some malfunctions. I didn't mean to offend."`
					goto lie
				`	"The Heliarchs asked me to do it."`
					goto truth
			label lie
			`	Around ten seconds pass without it saying anything, simply staring at your eyes and blinking slowly. "I see." Without another word, it walks away, headed to the Wardragon.`
				decline
			label truth
			`	"And why did you agree? Do you wish to join their ranks, perhaps?" It asks.`
			choice
				`	"I just wanted to help. It's not like I destroyed your ships or something."`
				`	"They told me that the Quarg keep others in 'separate boxes', and that you hinder their development."`
					goto hinder
			`	"Perhaps not, but you agreed to collect intel on us and our worlds, which hold countless civilians, and deliver the findings to a group that hates us."`
				goto end
			label hinder
			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
			label end

> 			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`

Author: Just depart and have that be that? Or depart and follow the player? I'm not sure on the latter since it'd be odd if it departed only to follow you to the Coalition and get killed.

> 			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`

Reviewer 3: No he shouldn't follow the player indeed, just make sure he leaves the system without any funny business.

> 			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`

Author: I did a thing, tell me if it doesn't work right.

> 			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`

Reviewer 3: Seems good, but maybe he should be staying too? 
Also I think he should have 
```to despawn 
  has "Heliarch Recon 3-A: offered"```
So that he doesn't stay around forever, staying in that system he can easily check if you're doing smth you shouldn't be, and its mostly to impress you probably that he goes in his Wardragon

> 			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`

Author: I gave the mission a deadline of 1 so that it fails pretty immediately, wouldn't that already despawn the ship?

> 			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs', human." It walks away, headed to the Wardragon it looked at before.`

Reviewer 3: You're right, I just didn't notice, it will indeed despawn the ship the next time you land on a planet

	landing
	to offer
		has "Heliarch Containment 4-A: done"
		not "Resistance: Questions: done"
	source "Remote Blue"
	destination "Station Cian"
	on offer
		conversation
			`The hundreds of Heliarch agents leave the three transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand.`
			`	You're informed by one of the local Heliarchs nearby that you should now head to <destination> to collect your payment of <payment>.`
				accept
	on complete
		payment 495000
		conversation
			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me, you must, Captain. On Fourth Shadow, broken out, another riot has, so organizing some ships from within the Ring of Power, I am."`
			`	He goes on to explain that the riot was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue riot, so the station is fairly devoid of a military presence now.`

mission "Heliarch Containment 5"
	name "Stop The Fleeing Ships"

> 	description "Defeat the criminal ships that are attempting to flee. Only disable the ships, because the Heliarch wish to find the source of their weapons."

Reviewer 3: You should add that you are meant to board them afterwards, in the description (the fact that they self destruct + I'm supposed to only disable them got me puzzled)

# Copyright (c) 2022 by Arachi-Lover
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 4: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```
The standard copyright header was changed recently.

# Copyright (c) 2022 by Arachi-Lover
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

mission "Heliarch Investigation 1"
	name "Transport Heliarch Agents"
	description "Bring <bunks> Heliarch agents to <destination>."
	minor
	passengers 3
	to offer
		random < 65
		"coalition jobs" >= 70
		has "Coalition: Contributor: done"
		not "Resistance: Questions: done"
	source
		near "Quaru" 4 10
	destination "Ring of Friendship"
	on offer
		conversation
			`A group of Heliarch agents rapidly approach you as you step onto the spaceport, looking rather agitated as they stand in front of you.`
			`	"Looking for work, the human visitor is?" the Saryd asks.`
			`	"To the <destination>, a means of transport we need," the Kimek follows.`

> 			`	"<payment>, we can offer," the Arach finishes.`

Reviewer 5: I think that they don't really need an introduction. The fact that a trio of heliarchs asks you to do something should be enough information. Also adds a bit of arrogance to the heliarchs.

> 			`	"<payment>, we can offer," the Arach finishes.`

Reviewer 6: If that's the standard we expect that's fine, it just felt affronting - so I am seeking to make sure that is intentional.

> 			`	"<payment>, we can offer," the Arach finishes.`

Author: It was intentional yeah.

mission "Heliarch Investigation 2"
	name "Heliarch Investigation"
	description "Take the Heliarch agents to the marked planets so that they can investigate signs of criminal activity."
	passengers 3
	to offer
		has "Heliarch Investigation 1: done"
		not "Resistance: Questions: done"
	source "Ring of Friendship"
	destination "Ring of Friendship"
	to complete
		has "HI2-MP: done"
		has "HI2-SF: done"
		has "HI2-SS: done"
		has "HI2-FS: done"
		has "HI2-IW: done"
		has "HI2-RB: done"
	on offer
		conversation
			`The Heliarch trio you brought here approaches you when you enter the spaceport, Kirrie being the first to greet you, "Captain <last>! Glad I am, that still around, you are."`
			`	"Accompany us, would you? A job in mind for you, our superior has." Iplora says.`
			choice
				`	"I don't see why not, lead the way."`
				`	"Not this time, I'm really busy now."`
					decline
			`	The trio escorts you through a number of restricted passageways, eventually leading you to a meeting room of sorts with about a dozen other Heliarchs, one of whom motions for you to sit with them.`

> 			`	You take a seat before one of the Kimek there speaks. "The honorable Captain, we salute. Make you feel welcome, I, Consul Iekie of the Heliarch council, hope we will."`

Reviewer 5: They could be keeping an eye on you specifically because the the player is the first human visitor in a while and shows some interest in helping the coalition. So the consuls could see great use in them.

> 			`	You take a seat before one of the Kimek there speaks. "The honorable Captain, we salute. Make you feel welcome, I, Consul Iekie of the Heliarch council, hope we will."`

Reviewer 6: True, but if there are any intermediate ranks it might be sensible for the Consul to have a "right-hand man" so to speak who is one rank down, who might handle any of the busy work associated with tracking and communicating with you.

> 			`	You take a seat before one of the Kimek there speaks. "The honorable Captain, we salute. Make you feel welcome, I, Consul Iekie of the Heliarch council, hope we will."`

Author: Consul is the highest rank.
This Intro introduces you to 3 of them, one in each ring, and each being representative of a different aspect of the Heliarch government.
Out of the 3, Iekie is the most involved with standard government business for the Heliarchy, but also is one of those in charge of investigating people and places that they believe might be involved with the Lunarium.

Considering that in one of the other chains where you meet a consul (Plortub on Ring of Power) you only ever meet at about halfway through the chain, and the last consul (Aulori on Ring of Wisdom) only ever shows up on another chain, *after* you've already done a lot of work for him via assisting his teams, I don't think it's an issue that Iekie, the more government-focused of the 3 consuls, is met early on on her chain.
You meet Iekie right at the start, Plortub at about the middle, and Aulori way later, on their respective chains.

I also don't want to go on creating stepping stone characters before the player can interact with the consuls proper, since it's mainly the consuls that the player will interact with during the campaign anyway.
The player is an exceptional figure in the Coalition, being an outsider, and having a JD of their own, so even if for normal aspiring Heliarchs meeting a consuls might be ways off, I don't think that should also apply to the player in these missions.

				accept
			label evidence1
			`	The agents then get closer to her before Iplora whispers something, handing her another report.`
			`	Iekie turns to you once more, saying, "Quite a find, this is. Done well, you have, Captain."`
				goto finalthanks
			label evidence2
			`	The agents then get closer to her before Iplora whispers something, handing her two separate reports.`
			`	Iekie turns to you once more, saying, "Most useful to our investigation, these findings will prove. Excellent work, Captain."`
				goto finalthanks
			label evidence3
			`	The agents then get closer to her before Iplora whispers something, handing her three separate reports.`
			`	Iekie turns to you once more, saying, "Enviable, your prowess at investigation is, <last>. Outstanding work, Captain."`
				goto finalthanks

mission "HI2-Bonus 1"
	invisible
	landing
	to offer
		has "Heliarch Investigation 2: done"
		"resistance evidence" == 1
	source
		government "Coalition"
	on offer
		payment 300000
		conversation
			apply
				clear "resistance evidence"
			`You receive a message from the Heliarch Consul Iekie as you go in for a landing.`

> 			`	"Captain <last>! Analyzed your findings, we have. Brought light to some stolen equipment, your aid has, and for that, grateful we are. Sending a monetary bonus as thanks, I am."`

Author: The theft of Heliarch equipment is made possible/facilitated by the combined efforts of:

- The Quarg, who relay to the Lunarium intelligence they get via stalking every Heliarch decision made in the rings,
- House Bebliss, the most advanced group in the Coalition in terms of understanding hyperlane communications, and the group responsible for mantaining such a network,
- House Idriss, the Arach House specialized on all things computers and electronics, and who are more than capable of jeopardizing most if not all Heliarch surveillance systems on a small area,
- Elirom, a former Heliarch Arbiter (second highest rank) who has a very good understanding of their modus operandi, and who can thus advise the Lunarium on the best way to go about their operations.

By comparison, the "minor rare accident" from your Friend of Arthropods PR is made possible/facilitated by:

- 

I would say that it's pretty consistent that a society that can prepare enough for the weather to the extent that they're essentially immune to such accidents, can still be susceptible to various major factions and former high ranking members working against them, so much so that equipment can be stolen and hidden away from such a society's government.

The Lunarium is getting aid from a whole lot of people. The weather accident in your PR is just that, an accident, pure chance. They're not at all comparable.
One of the alternatives I provided **is exactly making it not an accident, but instead product of drastic actions taken by the Heliarchs**.
I won't discuss that PR any more here, and I'd appreciate if you kept the discussion of your PR to your PR.

					goto reactor
				`	(Go further out and check the trees and bushes.)`
					goto trees2
				`	(Leave the rest of this to Kirrie.)`
			`	You apologize and tell Kirrie that you don't know what to do next. While he does agree to do it himself, he never stops asking for your opinion on his choices.`
			`	You spend a few more hours checking the warehouses until the rain and the wind get too dangerous for you to be moving from warehouse to warehouse, so you take shelter in one of them until the weather settles. Once it does, it is nearly nightfall, and Kirrie is notified by Iplora that she and Tugroob are already back at the ship. You two decide to head back as well.`
			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. Exhausted, they get back in their bunks, asking you to depart the planet.`
				accept
			label reactor
			apply
				"resistance evidence" ++
			`	Kirrie tries to open the door but can't get his card to work. "All the doors to these warehouses, my card should open. Damaged, the reader doesn't seem to be," he says. He decides that he must get in, and grabs a small sidearm, busting the door open with a couple shots.`

> 			`	At first glance there is nothing inside, but after he furiously crawls all over the place, he puts his head to a large metal panel on the opposite end of the room, before shooting that too. You approach him to find a hidden staircase leading underground.`

Author: I'm not sure I understand what you're getting at.
Kirrie is in a position where he now knows there's something wrong with this warehouse, his card didn't open it.
He knows he can get substantial back-up if needed, and knows that even if there's nothing in the warehouse, him making a mess there won't get him in any trouble.

> 			`	On your way down, Kirrie yells for you to take cover, and you barely escape a barrage of shots. Kirrie is hit on one of his legs before he can take cover.`

Reviewer 5: In my eyes, he is moving to cover and gets shot in the process, but reaches it anyways. but maybe using a sentence that makes that fact clearer is better.

> 			`	On your way down, Kirrie yells for you to take cover, and you barely escape a barrage of shots. Kirrie is hit on one of his legs before he can take cover.`

Author: Running to cover with you, but gets hit on one leg before making it there. Still gets to cover due to momentum/having 5 other legs.
Basically was just meant to portray that he didn't make it entirely unscathed, unlike the player. Will try to make it clearer.

> 			`	Just as you spot several small automatic turrets on the ceiling and walls, you two attempt to retaliate, but their fire proves too much. Barely escaping, you rush back outside, where Kirrie uses a small radio-like device to call for support. A pair of Heliarch Punishers soon land nearby, and dozens of armed agents enter the warehouse while you wait with Kirrie as his wound is treated.`

Author: Judicators have more free bunks, but Punishers can still hold more crew in total (110 vs 145).
This is more a product of these having been written before I even thought of doing the other Heliarch ships, though. Just wanted to have each of the agents call in the Heliarch ship that had the cockpit style of their species.

				goto fail
			label hole
			`	You two carefully make your way down the hole. It bottoms out around ten meters below the barn.`
			`	As you come down, Tugroob points to the cracked, rocky walls. He says that most of them were unstable, and you should watch out for rock slides.`
			`	Looking around, you spot several smaller holes on the walls just large enough for a human or Saryd arm. There are also few larger cracks that you two shouldn't have much trouble going through.`
			choice
				`	(Reach my arms inside the smaller holes to look for anything there.)`
				`	(Head for the larger cracks to see if they lead to anything.)`
					goto rockraider
			`	You carefully go about putting your arm inside a few of the holes, with Tugroob helping you up for the ones you couldn't reach, but ultimately you find nothing.`
			`	A few small rocks fell while you were doing so, but thankfully you and Tugroob are unscathed. Sand is piling up inside from the storm raging above.`
			`	"Dangerous, it will be, to remain here for long, Captain," Tugroob says.`
			choice
				`	(Head for the larger cracks in the wall now.)`

> 					goto rockraider

Author: I'll do the fixes myself, yeah.
Something something abandoned mining place something something mineshaft.

			`	After picking up Tugroob from the last factory, they all analyze their findings one last time. They do this by going over each individual factory, mechanically running down a long list of "possible abnormalities" for each, leaving a mark on the side, and briefly writing a description if any such "abnormality" was noted in a factory.`
			`	"Unauthorized power surges to some replicators, we found. Missing as well, some records of storage transport were," Iplora says. You land on a Heliarch base, and they hand all their findings to the agents there before returning to your ship. As they head to their bunks, you see several Heliarch ships fly off in the direction of some of the factories.`

mission "HI2-IW"
	name "Investigate <planet>"
	description "Transport the Heliarch trio to <destination> so that they can investigate signs of criminal activity."
	to offer
		has "Heliarch Investigation 1: done"
	source "Ring of Friendship"
	destination "Into White"
	to fail
		or
			has "Heliarch Investigation 2: declined"
			has "Heliarch Investigation 2: aborted"
			has "Heliarch Investigation 2 :failed"
	on complete
		conversation

> 			`Kirrie, already donning his winter coat, instructs you to land near the equator. He says him and Tugroob will conduct some subtle interrogations, stating that it's 'all they can do' in this climate.`

Reviewer 6: The physical climate is a limiting factor? Seems odd.

Author: I mean, the planet is very, very cold, and the Kimek and Arachi are known to not do well in such climate.
As for the clothing in spite of interrogations likely occurring indoors... well whenever my family and I traveled to a cold place, usually we wore the winter clothing even if we were indoors. Unless we had just gotten back to the hotel and knew we were only going to leave again in hours, we'd keep the clothes on.
I just don't think it's that unusual to wear winter clothes when on a planet as cold as Into White?

			choice
				`	"Good luck. Be careful not to get lost in all this snow."`
					goto warm
				`	"Hold up, I'll go with you so we can search faster."`
			`	She pauses on the ramp, looking back at you.`
			`	"Certain of that, are you, Captain? Treacherous, these storms can be."`
			choice
				`	"Don't worry, I can carry my weight through some snow."`
					goto snow
				`	"On second thought, you're right. Good luck out there."`
			label warm
			`	You slouch around your ship for a few hours, waiting for any signal from the Heliarch agents, but nothing comes up on your commlink. Iplora comes back, saying she couldn't find anything, so you leave and pick up Kirrie and Tugroob. They also return empty handed.`
				accept
			label snow
			`	She remains silent, and walks back inside. You close the hatch and prepare yourself for what's to come with the warmest clothing you could find. You open the hatch once more, and the both of you head out into the white fields. Not long after leaving your ship, you can make out small, dark lines nearby, probably pine trees. It seems you landed close to a forest. Unfortunately, they don't do much to protect you from the snowfall.`

> 			`	You continue deeper into the lifeless forest for what feels like hours, so long that your feet start feeling numb from the cold. After yet another turn, Iplora guides you out of the forest to snowy hills. The storm is not so terrible here, so you can faintly see cave entrances in some of the more distant hills. Looking back at the woods, you see that there's a small shack in the outskirts, not too far from where you are now.`

Reviewer 6: Cool, your clothing is warm. Is the outer layer waterproof? Cause if not it literally doesn't matter how warm your clothing is in this scenario.
Also, if you have proper footwear and attire, but not highly specialised footwear/attire this thing with the numb feet is about -20 centigrade, then the pain starts setting in around -26.

Author: Do I really need to mention that the player's winter clothing is waterproof? I thought it'd be pretty easy to infer they're not putting on a wooly sweater. Just seems like it'd be unnecessary.

For the numb feet thing, I personally never got it, but when we were on a place with snow for the first time, my father would complain about his feet going numb/hurting a bit. I imagine there's degrees to how numb your feet can feel due to cold weather, didn't mean to imply the harshest scenario with it.

			`	"Check the caverns, we must. More relentless, the storm will soon become," Iplora says.`
			choice
				`	"Lead the way."`
					goto bumpyroad
				`	"You go ahead. I'll see if I can find anything in that shack."`

> 			`	She nods, and starts making her way uphill. You start walking to the shack, but just a couple steps into it you trip on something hard, unlike the soft, deep snow you've been in up until now.`

Reviewer 6: Just FYI, right here, if your outer layer isn't waterproof - at this point you are now dead. Congratulations.

Author: Agreed.
But again, do I really need to specifically mention the player's winter clothing is waterproof?

			`	Iplora notices it and walks over to you, "Many boulders, this area has. Buried by the snow they have been. Great care you must take, while walking here," she explains before getting on with her own path to the caverns.`

> 			`	You reach the shack and knock to see if anyone is inside. Your question is answered as your soft knocks push open the door into an empty building. The place is abandoned, with only a few boxes laying around, small, empty bedrooms, and what seems to be a Kimek-styled fireplace, all covered in a thin layer of dust. A large carpet covers the center of the living room.`

Reviewer 7: How about this?
```suggestion
			`	You reach the shack and knock to see if anyone is inside. Your question is answered as your soft knocks push open the door into an empty building. The place is abandoned, with only a few boxes laying around, small, empty bedrooms, and a fireplace that resembles the spiky design of Kimek ships. All of them are covered in a thin layer of dust. A large carpet covers the center of the living room.`
```

			choice
				`	(Go inside with Iplora.)`
					goto cave
				`	(Search around the snow field.)`
			`	You tell her you'll look around the snow for a while and go inside if the storm gets rough again. Walking around in the blizzard, this time minding the bumpy ground below, you try to spot anything unusual in the snow. Although the scenery is a great sight, the task itself quickly becomes dull. Even after walking far away from the hills, your effort produces no results. The snow starts falling much faster, so you begin making your way back to the caverns.`
			`	With all the snow blocking your vision, you end up tripping over some of the mounds on your way and realize you don't exactly remember which cave Iplora entered. Considering the weather, though, taking shelter in any of them would be a good choice.`
			choice
				`	(Enter the nearest cavern.)`
					goto shelter
				`	(Try and dig through the snowy mounds to find something.)`
					goto dig
			label cave

> 			`	Iplora brings out a flashlight and lights up the way as you two explore the caverns. After some twists and turns she notices some large rocks, broken free from the walls, but still hugging them closely. She asks that you hold the flashlight for her and starts pushing the rocks, revealing a passage to a small campsite containing some makeshift beds, small wooden crates, and a worn down campfire in the middle.`

Reviewer 6: The wooden crates need less explanation here, but still, refer back to advanced society reference. Having them used at all could bear some explanation.

				accept
			label snowdig
			`	You start clawing your way down the snow as Iplora asks you to stop and continue moving toward the ship. You continue, your hands getting colder as the snow steals the heat even through your gloves. You hit a hard surface and start clearing from there, only to see it was, in fact, just a boulder.`
			`	"Satisfied? Return, we must. Come now," You turn to see her offering her hand once again.`
			choice
				`	(Take her hand.)`
					goto back
				`	(Head to another mound and try digging there.)`
			label keepdig
			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow muffling the sounds around you, you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, this time creating a faint metallic echo. She takes her hands off of you and lays down, now helping you dig out whatever is there. Once the crate is visible, she stands up. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`

> 			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast coming from the the top of the hills hits the ground near the hole. It misses you by just about a meter.`

Reviewer 6: Oh so now the crates are metal, also looks like a human coffin? If not, do Coalition species even use coffins? Would you know what one looked like even if they did?
Additionally, hooves are pretty good at snow, metal crates are the opposite of slippery when they're this cold, and if this crate is frozen you don't have a hope in hell of opening it with a crowbar, unless its internally powered somehow, in which case why would it need a crowbar to open?

Reviewer 5: It looks like a coffin to the player, I don't see Iplora saying anything about the shape

Author: From the player's perspective it appears to be a coffin, yeah.

			`	She yells for you to get down, pulling out her weapon and firing in the direction the shot came from.`
			apply
				"resistance evidence" ++
			choice
				`	(Take cover.)`
				`	(Pull out your gun and help her.)`
					goto help

> 			`	You get inside the hole, hoping you'll have enough protection from the blasts. You can still hear Iplora move around, and you occasionally peek your head over to see her shooting and taking cover, running and dodging incoming blasts. The shots stop and you see her come over, saying that it's safe to get out now.`

Reviewer 6: If you can dig through it with your hands, it won't. More likely a stray shot will collapse half a tonne of snow and ice onto you from the shoulders down. Congratulations you've just created a death trap for yourself.

			`	"Fled, our attacker has," she claims. From the looks of it she wasn't injured. You two get back to the crate.`
				goto opencrate
			label help

> 			`	You start shooting the hills above, trying to aim around where Iplora's own shots go to. A shot comes your way, and you jump to the side. Iplora takes cover inside the hole, firing a few more shots from there, then leaves up the hill. You follow her lead, approaching from another direction.`

Reviewer 6: How big is this hole? She's a centaur for pity's sake - with hooves that actually let her manoeuvre somewhat well. Also; see deathtrap.

Author: Big-ish?
Centaur or not, dodging shots like that implies more superpowers than good maneuverability.

I'll add some more cover to the snowfield that she could realistically go to.

			`	The shots alternate targets, some firing at Iplora, then some at you, then back at her. The blizzard is bad enough that you can't even see the attacker, but it seems that goes both ways, as you and Iplora manage to avoid getting hit without too much trouble.`
			`	You've barely gotten past the last cave you visited when the shots stop altogether. Iplora still takes aim for a bit, but ultimately lowers her weapon, coming toward you. "Fled, our attacker has," she claims. You two walk back to the crate.`
			label opencrate
			`	When it's finally open, you see it contains a Heliarch finisher torpedo. You two go around the fields, and find that many other mounds are identical crates. She uses a communication device and calls for Heliarch support. Several Heliarch Neutralizers arrive some time after the blizzard subsides, and you're directed to one of them to dry your clothes and make sure you're not frostbitten.`

> 			`	Once you leave, you see several Heliarch agents using proper equipment to dig out the crates and opening them dozens at a time, revealing several more torpedoes. A group has also made their way up the hills looking for any sign of whoever attacked you. Iplora speaks with some of the Heliarchs there, then tells you that they will handle everything from here. One of the ships takes both of you to where your own ship landed and leaves as you prepare to take off.`

Reviewer 6: So this is the glow - why do they glow?

Author: They have a glow in the outfit images.
Just thought it'd be neat imagery, but don't mind removing the glow bit.

			choice
				`	"Alright, which worlds do you need me to land on?"`
				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
					decline
			`	They ask you to show them your map of the galaxy, so you bring them inside your ship and point out to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`
			`	You give them a quick rundown of each Quarg world, and after discussing among themselves for a bit, the Saryd speaks. "Interested in the Quarg's mining operations, and the various construction stages of their ringworlds, the consul is."`
			`	He points to some of the worlds, and you list the places you need to land: <stopovers>.`
			`	They call for some engineers to come work on your ship's sensors, the whole thing taking just a few minutes. When they're done, the Heliarch agents thank you for your cooperation, and tell you to head for the <destination> when you've collected all the data.`
				accept
	to fail
		"reputation: Quarg" < 0
	on complete
		event "heliarch recon break" 10 21
		"coalition jobs" ++
		payment 165000
		conversation

> 			`The Heliarch trio are eagerly waiting for you when you return. They are quick to welcome you, urging you to show them your ship's logs so they can see what information you've gathered. They don't seem to pay much mind to data on Grakhord or Forpelog, being much more interested in the ringworlds. As they examine the data, though, the Saryd frowns, while the Kimek brings all his legs closer to his body, and the Arach squints all her eyes - probably their versions of that reaction.`

Reviewer 6: Realistically this would be the effort of hours upon hours to really look at the data - unless you're waiting around this should just be a note of what they seem most and least excited to have successfully got data on.

Reviewer 5: They knew what they were looking for and noticed that the data was missing, so I could see them quickly noticing the missing data. Maybe adding a stage of confusion when they first notice the data is gone

Author: The implication is that the Quarg just thwarted all attempts to gather data on the rings.
It doesn't take long to analyze the data when there's no data.

			`	"Too large and complex for a quick reading, planets and ringworlds are, so wish to get updated scans on Quarg ships, the consul now does," the Arach says.`
			`	"Scan them during our war, we of course did, but much more precise now, our technology has become," the Saryd explains. "To use one of our own scanners, we cannot ask of you. Much risk, it would pose, but if to the <planet>, a human outfit scanner you could bring, alter it to suit our purposes, we will."`
			choice
				`	"Alright, I'll go there once I have an outfit scanner ready."`
				`	"That sounds even more dangerous than what you had me do before. Sorry, but I'm not interested."`
					decline
			`	They thank you for your willingness to help, and tell you they will be waiting on the <destination>.`
				accept
	on complete
		require "Outfit Scanner"

mission "Heliarch Recon 2-B"
	name "Silent Scan"
	description "Now that the Heliarchs have altered your Outfit Scanner, head to <waypoints> and scan some Quarg ships there, then report back to <destination>."
	landing
	source "Ring of Wisdom"
	waypoint "Enif"
	to offer
		has "Heliarch Recon 2-A: done"
		not "Resistance: Questions: done"
	on offer
		conversation

> 			`The Heliarch agents who spoke to you are ready with a team of engineers once you land. They remove the Outfit Scanner from your ship and take it to some restricted area. After about an hour, they come back, and while you can't see any visible changes to the outfit, they assure you it is better suited for scanning Quarg ships now.`

Reviewer 6: Scanning equipment sensitivity is heavily dependent on design and component quality. They might be able to apply some software to see through interference or something, but if it comes to just scan accuracy they're going to have actually provide a coalition scanner, there's nothing they can do to your scanner that will help that.

Reviewer 5: They could have swapped out the components and installed a program that prevents your scanner from using its full power on anything other than a quarg ship. Then when you return they swap the original components back in.

Author: It's just meant to be them gradually getting you to get more information for them.
You go with an Outfit Scanner and they do some tweaking on it, you go and get the data.
They want more data, so you're tasked with using an actual Scanning Module next.

			choice
				`	"I was just curious about your species."`
					goto lie
				`	"What are you talking about? I never did any of that."`
					goto lie
				`	"Scanning? I'm sorry, I think my ship just had some malfunctions. I didn't mean to offend."`
					goto lie
				`	"The Heliarchs asked me to do it."`
					goto truth
			label lie
			`	Around ten seconds pass without it saying anything, simply staring at your eyes and blinking slowly. "I see." Without another word, it walks away, headed to the Wardragon.`
				accept
			label truth
			`	"And why did you agree? Do you wish to join their ranks, perhaps?" it asks.`
			choice
				`	"I just wanted to help. It's not like I destroyed your ships or something."`
				`	"They told me that the Quarg keep others in 'separate boxes', and that you hinder their development."`
					goto hinder
			`	"Perhaps not, but you agreed to collect intel on us and our worlds, which hold countless civilians, and deliver the findings to a group that hates us."`
				goto end
			label hinder

> 			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`

Reviewer 6: What if you haven't met all of these races?

Reviewer 5: Looking at the code You need to have visited the Korath and Hai ringworlds before Heliarch Recon 1 can be triggered which would reasonably give you some idea of the species. So the only species that applies to is the Hai and that can easily be fixed by adding a "First Contact: Hai" check to Heliarch Recon 1. Or adding an event were you talk to the Quarg on Alta Hai about the Hai when first landing there and using that as a check.

			`	"Take your time with this task, you of course can, Captain, but told us to ask you to be hasty, Consul Aulori has," the Arach says. "Unsure about how long this window of opportunity will last, all of us are."`
			choice
				`	"What's this 'window of opportunity' about?"`
				`	"What exactly is the task?"`
			`	"Normally, unavailable for us low ranks, this information would be," the Kimek begins. "But, instructed us on it, Consul Aulori has, so that relay the plan to you we could. Learned, the consuls have, that guarding a nearby archaeological site in human space, a lone Quarg Wardragon is. Aware of that, were you?"`
			`	You confirm it, explaining to them that you were involved in the work around the site on Zug, and were there when the Wardragon came in to protect it.`
			`	"Then no trouble finding it, you should have," the Arach says. "Far from other Quarg, that ship is, so easier for you to flee, it should be, if attack you it does."`
			`	"All we need, a quick scan is," the Saryd tells you. "Once scanned it, you have, come back right away, you should."`

> 			`	You're warned not to disable or destroy the Wardragon should it attack you, as that would no doubt permanently, damage your standing with them.`

Reviewer 7: ```suggestion
			`	You're warned not to disable or destroy the Wardragon should it attack you, as that would no doubt permanently damage your standing with them.`
```

mission "Heliarch Expedition 1"
	name "Extraterritorial Reconnaissance"
	description "Head through the wormhole in Deneb, land on the planet there, and enter the system with the broken ringworld to get data for the Heliarch."
	minor
	landing
	source "Ring of Wisdom"
	stopover "Ruin"
	waypoint "World's End"
	to offer
		has "Ruin: Landing: offered"
		or
			has "Heliarch Recon 3-B: done"
			has "Heliarch Recon 3-C: done"
	on offer
		conversation
			`Landing on the <origin> you remember that the Heliarchs have asked you to share with them any information you get on Quarg ships and ringworlds. Would you like to tell them about the destroyed ring you saw on the other side of the wormhole in Deneb?`
			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
				`	(No, they don't need to know.)`
					decline
			`	You flag down a group of Heliarch agents who were walking by, and they salute you. "A valuable friend, you've proven to be, Captain," a Saryd of the group says. "Of assistance, how can we be?"`
			choice
				`	"Can you bring me to a higher-up? I have some information on ringworlds."`
				`	"Is there anyone I can talk to about a broken ringworld?"`

> 			`	The group discusses among themselves for a bit, then they bring you to a restricted section of the ringworld, reserved for the Heliarch. "Wait here, please," they tell you, offering you a seat on one end of a long table.`

Reviewer 7: ```suggestion
			`	The group discusses among themselves for a bit, then they bring you to a restricted section of the ringworld, reserved for Heliarchs. "Wait here, please," they tell you, offering you a seat on one end of a long table.`
```

			`	Some ten minutes pass before anyone comes in, but after that many Heliarch members start filling up the room, taking their own seats at the table. When the door opens for the last of them to come in, the guard announces "Consul Aulori," to which all the Helliarchs stand up and salute, the noise of conversation giving room to hooves hitting the ground. He is an old Saryd of gray eyes, pale hairs, with a short, well-kept beard, wearing a peculiar necklace with four large gemstones hanging from it. He sits opposite to you, the other Heliarchs sitting back down when he does.`

> 			`	"Forgive me for the delay you must, colleagues," he says to the other Heliarchs. "Caught up in my evening prayers, I was." He examines the room for a while, briefly glancing at each of the Heliarchs, until his eyes settle on yours. "Told, I was, that some valuable information for us, you have, Captain <last>."`

Reviewer 6: Who in the everloving is a *Heliarch* praying to?

Author: There's the Pilgrimage mission chain, also with a Saryd, that goes some bits into Saryd religiousness and practices, if you want to check it out.

The Heliarchy is a meritocracy, yes. Whoever you are or what you believe doesn't matter to them, as long as you're loyal to the Heliarchy, and earn your rank.

			`	It all grows quiet for a while, and then the door opens again. Some of the other Heliarchs leave in a haste, and you're asked to head inside again. Consul Aulori is standing up still, giving you a nod as you come in.`
			`	"Adjusted, your ship's sensors will be. Better suited, they will become, for detailed tridimensional projections," he says. He has a guard bring you back to your ship, where some engineers go inside your ship and spend some time tinkering with it. After they're done, one of the Heliarchs that was in the room approaches you, telling you they need you to land on the planet you described, <stopovers>, and to travel to <waypoints> to scan the ringworld. He wishes you luck, then hurries back to the restricted areas.`
				accept
	on stopover
		dialog `You fly around the mysterious world for a while, until your sensors beep in an odd way, in what you guess is the signal that the Heliarch's modifications are done doing their read on the planet.`
	on enter "World's End"

> 		dialog `You fire up your ship's sensors, letting it collect the data the Heliarch want for hours on end. Eventually you hear a series of beeps in a dull rhythm, and look to see your their performance is back to normal.`

Reviewer 7: ```suggestion
		dialog `You fire up your ship's sensors, letting it collect the data the Heliarch want for hours on end. Eventually you hear a series of beeps in a dull rhythm, and look to see that their performance is back to normal.`
```

	on complete
		event "heliarch expedition break" 22 34
		payment 3000000
		conversation
			`You're met by Aulori when you land, as well as several other Heliarchs. You transmit your scan logs, and they bring you to a separate room once again, where a model of the broken ringworld is being projected for all to see.`

> 			`	"Skeptical of what you told us, we all were, Captain," Aulori says, gesturing for you to accompany him as he walks around the projection. "Claimed that insane, you were, some of those here did," he reaches down to tap a Kimek on her head, "Or that wasting our time with jokes, you were," he bumps another Saryd's arm, "Assured me, some of them had, that wholly impossible, it was, to break a ring," he pushes an Arach's leg with his hoof. He stops where the projection shows signs of greater damage, where a large chunk of the ring was cut off, littered with floating debris. "But not insane, neither a joker, you are. And impossible, this clearly is not. Invaluable information, what you've brought us is, Captain <last>. Investigate the site thoroughly, we must, so ask for your assistance again, we will, when a suitable crew for it, we have found."`

Reviewer 6: This display of calling people out by physical touch actively shames them in front of their peers and you. This is usually not politic unless one had a policy of radical transparency, which doesn't feel very Heliarch-like.
In fact, with things like the Srach great houses, and some of the things you've written about Saryd's, it wouldn't surprise me if that sort of behaviour led to untimely deaths, either of those shamed, or those doing the shaming.

			`	A Saryd rushes closer once Aulori says that, "Sir, against sending off anyone, the council is-".`
			`	"Agree, the council will," Aulori cuts him off, still looking at the projection.`
			`	The Saryd thinks for a while, then continues, "Take years, it would, to find and train someone willing to become an inspector. A dangerous task it-"`

> 			`	"And wait around for years, we will not," Aulori interrupts him again. "Either find someone willing, or find another line of work, you will," he looks at the Saryd, then at the door.`

Reviewer 6: I thought this was a 10 billion strong meritocracy. Where is this skill shortage from? Where is the preparation and planning?
Not to mention, where is this supremely blind arrogance inside their own ranks coming from? If the Heliarchy is half the meritocracy it claims to be then no one could have reached Aulori's position by being unable to listen to the well-informed advice of his relevant subordinates.

Author: Not meant to imply a skill shortage.
Leaving the Coalition is something that was prohibited by the Heliarchs a long time ago, and they've stopped sending out expedition parties of their own after the last all of them didn't make it back.

This isn't Aulori implying a skill shortage or ignoring feedback, it's Aulori implying that he doesn't care if this has been prohibited, he wants a thorough investigation on this ring, no matter what some other consuls might say.

I'll change up some bits of this chain to have it be more in line with him trying to keep this whole expedition section a secret, probably add another mission at the end too, after the Punisher has returned, where Aulori is in some discussion with other consuls about having done this in secret.

mission "Heliarch Expedition 4"
	name "Extraterritorial Venture"
	description "Escort a Heliarch ship through human space and into the wormhole in Deneb, so that they can further investigate the broken ringworld."
	landing
	source "Ablub's Invention"
	destination "Ruin"
	to offer
		has "Heliarch Expedition 3: done"
	on offer
		conversation
			`The Punisher's captain has some equipment loaded onto the ship, and the mechanics do one last maintenance check on the ship before it leaves home territory. The captain tells you they will follow your lead when you launch. Every other minute, he looks nervously at the sky.`
				accept
	npc accompany save
		personality escort heroic opportunistic
		government "Heliarch"
		ship "Heliarch Punisher (Fuel)" "Eldest Inspector"
	on visit
		dialog `You've reached <planet> but left the Heliarch ship behind. Best depart and wait for them to arrive.`
	on complete
		conversation
			`The Punisher's captain had you follow his ship all over the planet after they refueled in the planet's odd facilities. Apparently they want an isolated place to set up their camp here, somewhere they wouldn't be easily spotted.`

> 			`	They settle for a cavern on the way up a mountain, overlooking a foggy valley. A river is faintly visible through the dense fog. As they unload the cargo and start setting up, the captain comes to you and says that they'll meet you in the spaceport in a few hours to discuss "the acquisition of supplies."`

Reviewer 6: Actually: question, are they aware that for this entire story arc they're a) travelling through human space and quite visible, and b) visiting somewhere that humans could easily use one of their acquired jump drives to follow them and have a look at this advanced alien warship and crew that's just come by to nosey in their space?

Reviewer 5: Maybe add a quest where you fetch them a freighter they can modify and use as a ship in human space to this quest line.

Alternativly in the remnant cognizance storyline, it is established that replicas of every ship to have ever graced popular media are in use. So that excuse can maybe be reused.

Author: They don't really mind humans knowing about them, and though they really are worried about the possibility being attacked/followed, it's certainly not humans they're worried about.
I mean, I wouldn't worry about humans if I were in a Punisher.

	to offer
		has "Heliarch Expedition 4: done"
	on offer
		conversation
			`You've been waiting around for hours in the "spaceport" here when the Punisher finally comes back. The captain, a young Arach, comes to meet you, taking worried glances at the sky on his way. "Ready to do some shopping, are you, Captain?"`
			choice
				`	"Wasn't I gonna get some supplies for you? Figured you meant I just had to pick them up from Coalition space."`
				`	"Why do you even need supplies? Wasn't your cargo full with those crates?"`
			`	"A most... exquisite opportunity, this is. Very... excited, our researchers are. To keep them and the operation secure, I was tasked," he says. "Filled to the brim with only laboratory utensils and advanced equiment, our cargo was, unfortunately."`
			`	He goes over how they lack basic equipment to set up a proper camp and further details on how all the Punisher was carrying was the technology that will be used to study the ringworld, so they have no means of sustaining the camp here in the long run.`
			choice
				`	"You came here without supply or equipment? What the hell were you thinking?"`
					goto nosupply
				`	"Alright, what is it that you need?"`

> 					goto needs

Reviewer 6: Ah yes, the coalition can make sure to build a weather control system for a 1 in many million odds chance of a minor inconvenience but they can't supply their ship for an expedition properly, or communicate the need for a resupply chain to be established in advance?

Reviewer 5: A solution for this is that the researchers find the equipment missing (resistance stole the equipment), and ask you to fetch some replacements for them.

	name "Transport Heliarch Soldiers"
	description "Bring these <bunks> Heliarch soldiers, along with <cargo>, to <destination>, so they can contain a rioting group."
	minor
	deadline
	passengers 33
	cargo "military supplies" 25
	to offer
		has "Coalition: Contributor: done"
		has "main plot completed"
		"coalition jobs" >= 60
		random > 25
	source
		near "3 Spring Rising" 2 5
	destination "Fourth Shadow"
	on offer
		conversation
			`The spaceport on <origin> is in quite a lot of turmoil, as dozens of Heliarch agents march around at an accelerated pace. Some prepare to board their own ships, and others speak with some of the captains here.`
			`	It's no different with you. A Kimek agent approaching you after noticing you watching.`
			`	"Much assistance, we require, Captain," they say. "A transport for <destination>, we need. For <bunks> of us, and <cargo>. Of great urgency, this task is."`
			`	Without you even noticing it, an Arach agent has also gotten close, and after the Kimek is done speaking, says, "To contain a riot, there, we must go. <payment>, you will receive."`
			choice
				`	"Okay. Bring your friends and the cargo to my ship."`
					accept

> 				`	"That seems a bit excessive for a riot, no?"`

Author: Should I just cut it after "excessive" ?

> 				`	"That seems a bit excessive for a riot, no?"`

Reviewer 7: Yeah that would be good.

			`	A couple small boxes are brought inside, and upon opening them, Plortup shows you they contain several small drones - drones very similar to those employed by the repair modules used here.`
			`	"A union of technologies, it is. Of the scanning module, the incredible surveillance capabilities are. Of the repair drones, the versatile mobility and durable shells are."`
			`	He explains to you that you're to head to Secret Sky to pick up <cargo> produced on the planet, but as you're coming in for a landing, jettison the crates with the drones so that they may act as spybots.`
			`	"To <destination>, you must return, after completed this task, you have," Plortup says. "Your payment there, you will receive. Suffice, <payment> will, surely."`
			`	He wishes you good luck, and the trio from earlier escorts you back to your ship.`
				accept
	on stopover
		dialog `As instructed, you release the drones as you come in for a landing. Judging by the normal conditions of the spaceport, no one seems to have noticed anything as you pick up the cargo.`
	on complete
		payment 330000
		conversation
			`Landing on <planet>, you're met by Consul Plortup, who hands you your payment of <payment>.`

> 			`	"Aided us, once more, you have, Captain. To further work with you, I look forward. Search for me in the local spaceport anytime, you should. More work for you I most likely will have." He performs a brief salute, and leaves for some other section of the ring.`

Reviewer 7: ```suggestion
			`	"Aided us once more, you have, Captain. To further work with you, I look forward. Search for me in the local spaceport anytime, you should. More work for you I most likely will have." He performs a brief salute, and leaves for some other section of the ring.`
```

			`	"Ask for your assistance, once again, I must, Captain <last>," he says.`
			`	You have no time to ask what's going on, or refuse, as he has you brought to a separate room where a Saryd, Kimek, and Arach are waiting.`
			`	"Brewing on <destination>, another riot is. Your continued support, we hope you'll once more provide, Captain."`
			choice
				`	"Alright, what's the plan?"`
					goto plan
				`	"I'm not really comfortable in helping with these. Can you let me go?"`
			`	He perks his head up a bit, then mumbles something in his own language, momentarily turning off the translator. He gives in, and has someone bring you back to your ship.`
				decline
			label plan
			`	"Granted access to our civilian ships, you were. Three new Kimek Spires, you have bought. Many passengers to <destination>, in them you are carrying," he says.`
			`	The Saryd of the trio of captains then speaks up, "Pose as civilian ships, we must. Accompany your ship, we will."`
			`	"Armed and ready, the soldiers are. Follow your lead once you depart, we shall," the Kimek adds.`

> 			`	Plortup wishes you all good luck, and you follow the trio of captains back to your ship, seeing that the Spires are parked close to it.`

Reviewer 7: Don't think this bit is really necessary.
```suggestion
			`	Plortup wishes you all good luck, and you follow the trio of captains back to your ship.`
```

		ship "Kimek Spire" "Kiry Paichee"
		ship "Kimek Spire" "Achipa Kyrri"
		ship "Kimek Spire" "Pakee Pikary"
	on visit
		dialog `You've reached <planet>, but the Heliarch transports haven't arrived yet! Best depart and wait for them to get here.`

mission "Heliarch Containment 4-B"
	name "Undercover: Payment"
	description "Collect your payment for escorting the Heliarch transports."
	landing
	to offer
		has "Heliarch Containment 4-A: done"
		not "Resistance: Questions: done"
	source "Remote Blue"
	destination "Station Cian"
	on offer
		conversation
			`The hundreds of Heliarch agents leave the three transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand.`
			`	You're informed by one of the local Heliarchs nearby that you should now head to <destination> to collect your payment of <payment>.`
				accept
	on complete
		payment 495000
		conversation
			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me you must, Captain. On Fourth Shadow, broken out another riot has, so organizing some ships from within the Ring of Power, I am."`

> 			`	He goes on to explain that the riot was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue riot, so the station is fairly devoid of a military presence now.`

Reviewer 6: There's a lot of riots by this point, I don't know if it's addressed in the resistance side yet, but this would imply *widespread* unrest among the population with easily identifiable cause.

Author: Heliarchs wouldn't use the word riot, will change to attack or something like that so that it doesn't imply it's civilians.

mission "Heliarch Containment 5"
	name "Stop The Fleeing Ships"
	description "Defeat the criminal ships that are attempting to flee. You must disable and board their ships, to help the Heliarchs find the source of their weapons."
	landing
	to offer
		has "Heliarch Containment 4-B: done"
		not "Resistance: Questions: done"
	source "Station Cian"
	on offer
		"reputation: Lunarium" = -1000
		event "resistance fleeing rb"
		conversation

> 			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything in particular. You also notice that the job board is completely empty, that the market's trade values are not updating, and any news system is awfully destabilized.`

Reviewer 7: How about this?
```suggestion
			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything in particular. You also notice that the job board is completely empty, that the market's trade values are not updating, and any news system is non-functional.`
```

			`	Suddenly, a few Heliarchs rush past you heading for an office. They quickly reemerge and approach you.`
			`	"Captain! Helped us, you have. Trustworthy, you are, and in need of a combat-ready ship, we are."`
			choice
				`	"What's going on?"`
				`	"My ship isn't exactly combat-ready to fight your own warships."`

> 			`	They say that the station's systems haven't been functioning properly, and they haven't been able to contact neither the group on Remote Blue nor the fleet that left for Fourth Shadow.`

Reviewer 7: ```suggestion
			`	They say that the station's systems haven't been functioning properly, and they haven't been able to contact the group on Remote Blue, nor the fleet that left for Fourth Shadow.`
```

				accept
	npc accompany save
		personality escort heroic opportunistic
		government "Heliarch"
		fleet
			names "heliarch"
			fighters "heliarch fighter"
			variant
				"Heliarch Judicator" 2
				"Heliarch Pursuer" 4
				"Heliarch Rover" 4
				"Heliarch Stalker" 4
	on visit
		dialog `You have reached <planet>, but you left some of the Heliarch escorts behind. Better depart and wait for them to get here.`
	on complete
		"coalition jobs" ++
		payment 150000
		conversation
			`Lotuora pays you <payment> when you land. "An idea for the next drill, I've had," she says. "Need to arrange some more ships, I will, so some hours, I will need. Meet me in the spaceport when I'm done, you should."`

mission "Heliarch Drills 2-A"
	name "Pursuit Drill"
	description "Land on the <planet> while avoiding the Heliarch Hunters."
	source "Belug's Plunge"
	destination "Shadow of Leaves"
	to offer
		has "Heliarch Drills 1: done"
		not "Resistance: Questions: done"
	on offer
		conversation

> 			`When you find Magister Lotuora in the spaceport, she shows you to a map of the Coalition, pointing to two systems, one in Kimek and one in Saryd space. "Made many calls, I have, and to assemble many ships for this drill, I've managed. How fit for their job, our Hunters are, we shall see."`

Reviewer 7: ```suggestion
			`When you find Magister Lotuora in the spaceport, she shows you to a map of the Coalition. Pointing to two systems, she indicates one in Kimek and one in Saryd space. "Made many calls, I have, and to assemble many ships for this drill, I've managed. How fit for their job, our Hunters are, we shall see."`
```

			choice
				`	"What do you mean?"`
					goto explain
				`	"Will I be fighting them?"`
			`	"No, no fighting," she says. "Not yet. Only simulating a pursuit, they will be."`
			label explain
			`	She points to the systems again, "To land on <planet> and Ashy Reach unseen by the Hunters, your task is. Attack you, they will not, but allow them to be in the system when you land, you cannot. Once cleared both planets, you have, return here for 300,000 credits, you may."`
			choice
				`	"Ok, I'll head out to the planets then."`
					goto accept
				`	"Can you tell me where my pursuers will be?"`
			`	"Know of their pursuers, the pursued do not," she says, letting a laugh as if it was a joke. "Well, usually... From many systems across the Coalition, the Hunters are. Ready to follow your ship when you launch, they will be."`
			label accept

> 			`	She points again to one of the planets, "Still preparing, some Hunters around Ashy Reach are, so head for <planet> first, you should," she wishes you luck on avoiding the Heliarch Hunters, and heads off to a Heliarch military building.`

Reviewer 7: ```suggestion
			`	She points again to one of the planets, "Still preparing, some Hunters around Ashy Reach are, so head for <planet> first, you should." She wishes you luck on avoiding the Heliarch Hunters, and heads off to a Heliarch military building.`
```

		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Sol Kimek"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "3 Spring Rising"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "5 Summer Above"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "3 Pole"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "12 Autumn Above"
	on visit
		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`

mission "Heliarch Drills 2-C"
	landing
	name "Pursuit Drill"
	description "Land on the <planet> while avoiding the Heliarch Hunters."
	source "Ashy Reach"
	destination "Belug's Plunge"
	to offer
		has "Heliarch Drills 2-B: done"
		not "Resistance: Questions: done"
	on offer
		conversation

> 			`Once again, you managed to avoid the Heliarch Hunters while landing on <origin>, and once again a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me Captain. Forgotten something, I had. Heard of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`

Reviewer 7: ```suggestion
			`Once again, you managed to avoid the Heliarch Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me Captain. Forgotten something, I had. Heard of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```

			variant
				"Heliarch Hunter"
		system "Speloog"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Torbab"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Flugbu"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Debrugt"
	on visit
		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`
	on complete
		"coalition jobs" ++
		payment 470000
		conversation
			`Magister Lotuora meets you when you land, handing you your payment of <payment>, and congratulating you for managing to evade the pursuing ships. "Frustrated many of the Hunter pilots, I heard you have," she says with a laugh. "Good training for them, I hope this was. Finishing up the arrangements for one more drill, I am. When ready, you are, meet me in the spaceport, you must."`

mission "Heliarch Drills 3"
	name "Combat Drill"

> 	description "Head to the <waypoints> system to fight the <npc>, reporting back to <planet> when you succeed. You must only disable the ship, destroying or capturing it would deem you an enemy of the Heliarchs."

Reviewer 7: ```suggestion
	description "Head to the <waypoints> system to fight the <npc>, reporting back to <planet> when you succeed. You must only disable the ship. Destroying or capturing it would deem you an enemy of the Heliarchs."
```

	source "Belug's Plunge"
	waypoint "Torbab"
	to offer
		has "Heliarch Drills 2-C: done"
		not "Resistance: Questions: done"
	on offer
		conversation
			`Lotuora is speaking with a Kimek Heliarch captain at the entrance to the spaceport. "Ah, there the Captain is," she tells the Kimek. "Captain <last>, your sparring partner, this is," she says, gesturing to the Kimek, who salutes you with a short bow of her head. "Excited to test her ship against an outsider, she is."`

> 			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, boarding a Punisher and flying away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall."`

Reviewer 7: ```suggestion
			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, where she boards a Punisher and flies away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall."`
```

			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`
				accept
	npc disable save
		government "Heliarch Test Dummy"
		personality disables staying nemesis heroic
		system "Torbab"
		fleet
			names "heliarch"
			variant
				"Heliarch Punisher (Scrappy)"
	on fail
		"reputation: Heliarch" = -1000
		"reputation: Coalition" = -1000
	on complete
		"coalition jobs" ++
		"assisted heliarch" ++
		payment 800000
		conversation

> 			`Magister Lotuora greets you when you return. "Prevailed over one of our Punishers, you have! Perhaps the real threat, you are, not the Quarg," she says, waiting for a few seconds as she looks at you, as if expecting you to laugh. She hands you <payment>, and continues, "Sent a recording of the fight, I was. Learn much from it, I hope we can."`

Reviewer 7: ```suggestion
			`Magister Lotuora greets you when you return. "Prevailed over one of our Punishers, you have! Perhaps the real threat, you are, not the Quarg," she says, looking at you for a few seconds as if expecting you to laugh. She hands you <payment>, and continues, "Sent a recording of the fight, I was. Learn much from it, I hope we can."`
```

> 			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy, and wishes you safe travels, heading to a Heliarch building.`

Reviewer 7: ```suggestion
			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She finally wishes you safe travels, heading to a Heliarch building.`
```

mission "Heliarch License"
	name "Meet With The Consuls"
	description "Head to the <destination>, where the Heliarch consuls will make you a Heliarch agent."
	minor
	landing
	source
		near "Quaru" 1 100
	destination "Ring of Friendship"
	to offer
		or
			"assisted heliarch" == 5
			and
				"coalition jobs" >= 90
				"assisted heliarch" == 4
			and
				"coalition jobs" >= 120
				"assisted heliarch" == 3
		not "Resistance: Questions: done"
	to fail
		has "Resistance: Questions: done"
	on offer
		conversation
			`When you land on <origin>, you find a Heliarch delegation waiting for you outside. When you meet them, the Saryd of the group says, "A great ally to our Coalition, you have proven to be."`
			`	"Discussed your exploits, the consuls have. Much contentment, your contributions brought us," the Kimek continues.`
			`	"With you, a meeting they request. To <destination> you must go, if to join our ranks, you so wish," the Arach finishes.`
				accept
	on complete

> 		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology, and a translation device, to freely converse with the Coalition species."

Reviewer 7: ```suggestion
		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology. Was also given a translation device, to freely converse with the Coalition species."
```

		set "language: Coalition"
		event "first heliarch unlock"
		set "license: Heliarch"
		conversation
			`When you land, you're met with a bit of ceremony. A trio of Heliarch consuls welcomes you, each carrying a gemstone: an emerald for the Saryd, a ruby for the Kimek, and a sapphire for the Arach. They each pass the precious stones to you, starting with the Saryd, then the Kimek, and lastly the Arach. Cameras and photographers surround the scene, kept at bay by lines of Heliarch guards. It seems your joining the ranks has turned into something of an event.`
			`	When the consuls are done, they have you follow them into the restricted sections of the ring, bringing you to a very large, circular room, somewhat like a colosseum or an auditorium, with hundreds of Heliarch consuls seated. There are also many more camera workers and photographers, though all of them are Heliarch members this time. You're instructed to head to the center.`

> 			`	The consuls who brought you here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much, this outsider has, shown devotion to our Coalition, this outsider has."`

Reviewer 7: ```suggestion
			`	The consuls who brought you here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much, this outsider has, and shown devotion to our Coalition, this outsider has."`
```

			`	"From humanity, <first> <last> has come," the Kimek continues. "Our most recent neighbor, and a group still under the chiding eyes of the Quarg, they are. To reach them all, to free them all, we still cannot, but, our first step in that front, Captain <last>'s coronation is."`
			`	The Saryd approaches you, carrying some type of chest. Opening it, she reveals a golden circlet inside, staying silent and looking at you.`
			choice
				`	"Do I put it on myself?"`
				`	(Take the circlet.)`
					goto circlet
			`	"Allow others to crown us, we do not," she says. "Not anymore."`
			choice
				`	(Take the circlet.)`
			label circlet
			`	You pick up the circlet, and raise it to your head, the Heliarch consuls all standing up when you do. You bring it down and adjust it to your head. Once the "crowning" is done, many of the consuls cheer in celebration, with something akin to a round of applause, and more of the speeches from before.`

> 			`	When it's all over, you're escorted back to your ship, where you're told that the circlet serves as a permission to buy certain outfits. They also hand you a small box, a translation device. "One of us you now are, Captain," the Saryd says. "Find some new work in our job boards, you may, if properly equipped, you are."`

Reviewer 7: ```suggestion
			`	When it's all over, you're escorted back to your ship, where you're told that the circlet serves as a permission to buy certain outfits. They also hand you a small box, which you soon realize is a translation device. "One of us you now are, Captain," the Saryd says. "Find some new work in our job boards, you may, if properly equipped, you are."`
```

# Copyright (c) 2022 by Arachi-Lover
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

mission "Heliarch Investigation 1"
	name "Transport Heliarch Agents"
	description "Bring <bunks> Heliarch agents to <destination>."
	minor
	passengers 3
	to offer
		random < 65
		"coalition jobs" >= 70
		has "Coalition: Contributor: done"
		not "Resistance: Questions: done"
	source
		near "Quaru" 4 10
	destination "Ring of Friendship"
	on offer
		conversation

> 			`A group of Heliarch agents rapidly approach you as you step onto the spaceport, looking rather hasty as they stand in front of you.`

Reviewer 8: Is 'hasty' the right word here? 'Flustered' or 'agitated' maybe?

Author: Changed to agitated.

> 			`	"Looking for work, the human visitor is?" The Saryd asks.`

Reviewer 8: This is perhaps a stylistic choice, but I think that when these guys are talking with their reversed syntax it doesn't need a comma. I think that if an English-speaker said it this way a comma would be used because we are essentially going back and filling in a blank that was missed, but for these guys this is just the way they speak.

"Looking for work the human visitor is?"

I have only mentioned it once because I know that it would be pretty annoying to have a million comments all basically saying the same thing, especially if that one thing has a good chance of being over-ruled. :-)

Author: I might have used a lot more commas than usual for the Intros as a good portion of them was written years ago, but Coalition speech in general has a lot of commas used, so won't be dropping most of them.

			`	"To the <destination>, a means of transport we need," the Kimek follows.`
			`	"<payment>, we can offer," the Arach finishes the proposal.`
			choice
				`	"Of course, I'll show you to my ship."`
				`	"Sorry, I'm not looking for any work currently."`
					decline

> 			`	They thank you for agreeing to deliver them to the <planet>, and introduce themselves: the Saryd is called Iplora, the Kimek is Kirrie, and lastly the Arach is known as Tugroob. You help them settle in their bunks, wondering why they seemed so rushed.`

Reviewer 8: ```suggestion
			`	They thank you for agreeing to deliver them to <planet>, and introduce themselves; the Saryd is called Iplora, the Kimek is Kirrie, and lastly the Arach is known as Tugroob. You help them settle in their bunks, wondering why they seemed so rushed.`
```

Author: `<planet>` here is being replaced with "Ring of Friendship," and before replacements where the planet in question is one of the rings, a "the" is put in place so that it's always "the [Ring of something]."

				accept
	on complete
		"coalition jobs" ++
		payment 90000
		conversation

> 			`The three Heliarchs leave your ship, heading for some section of the spaceport, but not before Iplora hands you <payment>. "Speak with our superior, we must. Need your services again, we likely will, so if interested in further work, you are, look for us in the spaceport, you should."`

Reviewer 8: ```suggestion
			`The three Heliarchs disembark, heading immediately across the spaceport, but not before Iplora hands you <payment>. "Speak with our superior, we must. Need your services again, we likely will, so if interested in further work, you are, look for us in the spaceport, you should."`
```
(I know I said in the previous comment that I would only mention it one time, but this bit of speech is a perfect example of why I think we should drop commas, see how it breaks up the flow of the text?)

				`	(Check the warehouse with the leaky roof.)`
					goto reactor
			`	Kirrie uses his card to open the door, explaining that it is an access card granted to Heliarch agents, in the event they need to conduct any work such as this, or simply to identify themselves as an agent. Unfortunately the warehouse turns out to contain nothing of any significance. It's storage for what Kirrie says are toy parts. Just mountains of plastic stacked together. You check a few more like this, but none turn out to have anything, and as the rain and wind become too much to walk through, you take shelter in one of the warehouses.`
			`	Once the weather settles near nightfall, Iplora notifies him that she and Tugroob are already back at the ship, so you two decide to return as well.`
			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
				accept
			label trees2
			`	You suggest looking for something unusual near the plant life you spotted while the rain is still fairly tame, and he follows you. You spend a few minutes looking around the trees but find nothing suspicious.`

> 			`	While passing by another set of bushes, he notices that some of them are awkwardly tilted. Upon further inspection, he tells you that they appear to have been removed, and then forcibly put on the ground again, noting the odd, wavy patterns of the small rocks around the roots. The rain has gotten heavier.`

Reviewer 8: ```suggestion
			`	While passing by another set of bushes, Kirrie notices that some of them are awkwardly tilted. Upon further inspection, he tells you that they appear to have been removed, and then hastily replanted, noting the odd wavy patterns of the small rocks around the roots. The rain has gotten heavier.`
```

			choice
				`	(Dig out the bushes.)`
					goto bushending
				`	(Head back to the open warehouses and take shelter there.)`

> 			`	Though you can barely hear each other in the rain, Kirrie insists on staying there and digging through the ground for a while to see if he finds anything.`

Reviewer 8: ```suggestion
			`	Though you can barely hear each other in the rain, Kirrie insists on staying there and digging through the ground for a while to see if he can find anything.`
```

			`	Once the weather settles near nightfall, Iplora notifies him that she and Tugroob are already back at the ship, so you two decide to return as well.`
			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
				accept
			label decline
			`	Kirrie leaves you behind and you sit there on your own for a number of hours. Eventually, the trio returns, revealing that only Tugroob had found something, mentioning some evidence of smuggled weapons passing through the planet.`
			`	The three agents get back in their bunks and say you may depart now.`

mission "HI2-SF"
	name "Investigate <planet>"
	description "Transport the Heliarch trio to <destination> so that they can investigate signs of criminal activity."
	to offer
		has "Heliarch Investigation 1: done"
	source "Ring of Friendship"
	destination "Stronghold of Flugbu"
	to fail
		or
			has "Heliarch Investigation 2: declined"
			has "Heliarch Investigation 2: aborted"
			has "Heliarch Investigation 2 :failed"
	on complete
		conversation

> 			`While you doubt anyone opposing the Heliarchs would find much success in this massive compound, it seems the resources, ships, and strategic location the planet holds made it be marked for investigation anyway.`

Reviewer 8: ```suggestion
			`While you doubt anyone opposing the Heliarchs would find much success in this massive compound, it seems the strategic location held by the planet, and the volume of resources and ships, marked it for investigation anyway.`
```

			`	You follow Tugroob to the upper levels but once again find nothing but dusty containers. A small gap between the planks here allows some sand to come inside. The storm is getting closer.`
			`	"Much too obvious a hiding spot, this would be," he says.`
			choice
				`	"You're right. Should we go outside and check around that hole?"`
					goto hole2
				`	"Maybe, but they could be trying to hide in plain sight. Let's open up these crates."`
			`	You two go about opening all the crates on the upper floor but end up finding nothing.`
			`	"Find nothing here we will, Captain" Tugroob says.`
			choice
				`	"Sorry, let's go outside and check that hole while we can."`
					goto hole3
				`	"Hold on. We can still check the ones down there."`
			`	At this point, Tugroob starts using all his body weight while gripping the containers with all legs at the same time, crushing each crate and barrel open. Once everything is broken up and it turns out there was nothing inside any of them after all, you two try to leave, but with the sandstorm raging strong, you decide to wait it out inside the barn.`

> 			`	Night falls, with no sign of the storm dying down, and after receiving a notification from Iplora, Tugroob informs you that you should both head back to the ship and pick her and Kirrie up. You two meet up with the rest of the group, and the agents head to their bunks.`

Author: Added a short bit on that, let me know if it needs some more to it.
Also, rearranged these repeated instances of the failed ending into a label, with gotos where repeated cases were.

> 			`	Night falls, with no sign of the storm dying down, and after receiving a notification from Iplora, Tugroob informs you that you should both head back to the ship and pick her and Kirrie up. You two meet up with the rest of the group, and the agents head to their bunks.`

Reviewer 7: Looks good.

			choice
				`	(Reach your arms inside the smaller holes to look for anything there.)`
				`	(Head for the larger cracks to see if they lead to anything.)`
					goto rockraider
			`	You carefully go about putting your arm inside a few of the holes, with Tugroob helping you up for the ones you couldn't reach, but ultimately you find nothing.`
			`	Many small rocks fell while you were checking the holes, but miraculously you both remain unscathed. Unfortunately, too much sand from the storm above is piling up for you to look around for more holes or the exact location of the larger cracks you saw before.`
			`	"Dangerous, it will be, to remain here for long, Captain," Tugroob says as he urges you to cling to them. He carefully crawls back up, and you two take refuge inside the barn.`
			`	Night falls, with no sign of the storm dying down, and after receiving a notification from Iplora, Tugroob informs you that you should both head back to the ship and pick her and Kirrie up. You two meet up with the rest of the group, and the agents head to their bunks.`
				accept
			label decline

> 			`	"I see," he says, then give you the coordinates they will go to. You drop them off and return to the hangars.`

Reviewer 8: Inconsistent pronouns; is it 'he' or 'they'? Assuming 'they';

```suggestion
			`	"I see," they say, then give you the coordinates they will go to. You drop them off and return to the hangars.`
```

Author: It's 'he'. Any 'they' instances here are because I missed them when editing this mission.

			`	You continue deeper into the lifeless forest for what feels like hours, so long that your feet start feeling numb from the cold. After yet another turn, Iplora guides you out of the forest to snowy hills. The storm is not so terrible here, so you can faintly see cave entrances in some of the more distant hills. Looking back at the woods, you see that there's a small shack in the outskirts, not too far from where you are now.`
			`	"Check the caverns, we must. More relentless, the storm will soon become," Iplora says.`
			choice
				`	"Lead the way."`
					goto bumpyroad
				`	"You go ahead. I'll see if I can find anything in that shack."`
			`	She nods, and starts making her way uphill. You start walking to the shack, but just a couple steps into it you trip on something hard, unlike the soft, deep snow you've been in up until now.`
			`	Iplora notices it and walks over to you, "Many boulders, this area has. Buried by the snow they have been. Great care you must take, while walking here," she explains before getting on with her own path to the caverns.`

> 			`	You reach the shack and knock to see if anyone is inside. Your question is answered as your soft knocks push open the door into an empty building. The place is abandoned, with only a few boxes laying around, small, empty bedrooms, and what seems to be a Kimek-styled fireplace, all covered in a thin layer of dust. A large carpet covers the center of the living room.`

Author: Bunch of spikes, just like their ships? Don't recall if I ever had something in mind on that. Could just drop the mention of it being Kimek-styled, but could add in a suggested description if you have one.

			`	You trip over a few more of the small mounds, and right in the middle of the snowfield, you start wondering which cavern Iplora is in now. Though you can't be sure if you'll find her by entering the nearest one, it would certainly be better than standing outside in what will soon be a blizzard.`
			choice
				`	(Enter the nearest cavern.)`
					goto shelter
				`	(Try and dig through the snowy mounds to find something.)`
			label dig
			`	You claw away at the snow and hit a hard surface, but it turns out to be just a rock. You try some other mounds to no avail, and right as you finish digging through another pile of snow only to find another rock, Iplora appears behind you.`
			`	"Counterproductive, cleaning away the snow is," she says. "Found nothing, I have. Back to your ship, we must go."`
			choice
				`	(Go back to the ship.)`
					goto back
				`	(Keep digging.)`
					goto keepdig
			label shelter

> 			`	You rush inside the nearest cave, waiting for the weather to clear up a bit. After a while you notice a light coming from further inside the cave, and Iplora emerges soon after. "Found anything?" She asks, and you shake your head. Aside from some small campsites, she didn't either. You wait inside a bit more, and when the weather clears and night approaches, you follow her back to the ship.`

Reviewer 7: ```suggestion
			`	You rush inside the nearest cave, waiting for the weather to clear up a bit. After a while you notice a light coming from further inside the cave, and Iplora emerges soon after. "Found anything?" she asks, and you shake your head. Aside from some small campsites, she didn't either. You wait inside a bit more, and when the weather clears and night approaches, you follow her back to the ship.`
```

			`	Once you're there, you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
				accept
			label bumpyroad
			`	Following her up the hills proves difficult at first, with your legs sinking into the snow, but after a few minutes the ground below becomes more and more solid - if a bit bumpy. Iplora explains that the area has several small and medium sized boulders buried by the snow over the years. You try to watch your step, but as the snowfall grows stronger, you trip over one of the mounds. She helps you up, and the two of you eventually arrive at the entrance of one of the caverns.`
			choice
				`	(Go inside with Iplora.)`
					goto cave
				`	(Search around the snow field.)`

> 			`	You tell her you'll look around the snow for a while and go inside if the storm gets rough again. Walking around in the blizzard, this time minding the bumpy ground below, you try to spot anything unusual in the snow. Although the scenery is a great sight, the task itself quickly becomes dull, and even after walking far away from the hills, shows no results. The snow starts falling much faster, so you begin making your way back to the caverns.`

Reviewer 7: ```suggestion
			`	You tell her you'll look around the snow for a while and go inside if the storm gets rough again. Walking around in the blizzard, this time minding the bumpy ground below, you try to spot anything unusual in the snow. Although the scenery is a great sight, the task itself quickly becomes dull, and even after walking far away from the hills, your effort produces no results. The snow starts falling much faster, so you begin making your way back to the caverns.`
```

			`	You search through the crates as she examines the room looking for some other hidden area, but neither of you find anything. She checks the campfire and concludes that nobody has been here for a few months. You continue exploring the cave and eventually loop back outside, leaving via another cave entrance, this time much further from the forest according to Iplora. You can't even see the trees anymore, the blizzard has gotten much stronger.`
			choice
				`	(Suggest returning to the ship.)`
					goto bumpyblizzard
				`	(Continue looking around the caves.)`
			`	You head back inside and continue exploring the cave system. You two find a few more of the hidden campsites, but all are as abandoned as the first one. You're alerted by the wind, snow, and ice whenever you're close to another exit.`

> 			`	After a few hours, the storm finally subsides, and you both return to your ship. you leave to pick up Kirrie and Tugroob, who also come back empty handed.`

Reviewer 8: ```suggestion
			`	After a few hours, the storm finally subsides, and you both return to your ship. You leave to pick up Kirrie and Tugroob, who also come back empty handed.`
```

				accept
			label bumpyblizzard
			`	Iplora hesitates for a bit, then explains that you would probably get sick going through the storm now. She suggests you wait in the cave until it is safe for human passage.`

> 			`	You insist, and she reluctantly gives in, telling you to stick close to her. She moves slowly, so you have no problem keeping up, but time and again you keep falling over, tripping on the mounds the rocks have created. Another fall, and Iplora reaches her hand out to help you up once again.`

Reviewer 7: ```suggestion
			`	You insist, and she reluctantly gives in, telling you to stick close to her. She moves slowly so you have no problem keeping up, but time and again you keep falling over, tripping on the mounds the rocks have created. Another fall, and Iplora reaches her hand out to help you up once again.`
```

			choice
				`	(Take her hand.)`
				`	(Try and dig through the snow to find something.)`
					goto snowdig
			label back
			`	She helps you up, and you continue on your way down the hills. You get back to the forest, and she starts finding the trail back to the ship. Halfway through, the storm gets progressively weaker.`

> 			`	When you two return to your ship. you leave to pick up Kirrie and Tugroob, who also come back empty handed.`

Reviewer 8: ```suggestion
			`	When you two return to your ship, you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
```

				accept
			label snowdig
			`	You start clawing your way down the snow as Iplora asks you to stop and continue moving toward the ship You continue, your hands getting colder as the snow steals the heat even through your gloves. You hit a hard surface and start clearing from there, only to see it was, in fact, just a boulder.`
			`	"Satisfied? Return, we must. Come now," You turn to see her offering her hand once again.`
			choice
				`	(Take her hand.)`
					goto back
				`	(Head to another mound and try digging there.)`
			label keepdig

> 			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, and a faint metallic sound echoes. She takes her hands off of you and lays down, now helping you dig out whatever is there. She stands up once the crate is visible. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`

Reviewer 7: ```suggestion
			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow muffling the sounds around you, you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, this time creating a faint metallic echo. She takes her hands off of you and lays down, now helping you dig out whatever is there. She stands up once the crate is visible. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`
```

> 			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast, coming from the the top of the hills, hits the ground near the hole, missing you by just about a meter.`

Reviewer 7: ```suggestion
			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast, coming from the the top of the hills, hits the ground near the hole. It misses you by just about a meter.`
```

			`	She yells for you to get down, pulling out her weapon and firing in the direction the shot came from.`
			apply
				"resistance evidence" ++
			choice
				`	(Take cover.)`
				`	(Pull out your gun and help her.)`
					goto help
			`	You get inside the hole, hoping you'll have enough protection from the blasts, and hear Iplora move around, occasionally peeking to see her shooting and taking cover, running and dodging incoming blasts. The shots stop, and you see her come over, saying you can get out now.`
			`	"Fled, our attacker has," she claims. From the looks of it she wasn't injured. You two get back to the crate.`
				goto opencrate
			label help
			`	You start shooting the hills above, trying to aim around where Iplora's own shots go to. A shot comes your way, and you jump to the side. Iplora takes cover inside the hole, firing a few more shots from there, then leaves, going uphill. You follow her lead, approaching from another direction.`

> 			`	The shots alternate targets, some heading to Iplora, then some to you, then back at her. The blizzard is bad enough that you can't even see the attacker, but it seems that goes both ways, as you and Iplora manage to avoid getting hit without too much trouble.`

Reviewer 8: ```suggestion
			`	The shots alternate targets, some firing at Iplora, then some at you, then back at her. The blizzard is bad enough that you can't even see the attacker, but it seems that goes both ways, as you and Iplora manage to avoid getting hit without too much trouble.`
```

			choice
				`	"I'm not sure I understand what you're getting at."`
				`	"Obviously I have my limits as for what I would do."`
			`	"A jump drive, you possess. Far away, you can travel. Bother you, the harsh cages of the Quarg do not," the Saryd explains as all three walk toward your ship.`
			`	"Few in number, our own jump drives are, and too risky would it be, for one of our own to venture far out," the Kimek adds.`
			`	"If to help us gain knowledge of the opressors, you wish, lend the Coalition your ship's sensors for a short time, would you?" Asks the Arach.`
			choice
				`	"Are you asking me to spy on the Quarg?"`
				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
					decline
			`	"As espionage, a 'malfunction' qualifies not," the Saryd responds.`
			`	"Merely alter your own ship's sensors, we would. For a time, specially tuned to scan Quarg worlds, they would be," the Kimek clarifies. "No evidence against you, they would have."`
			choice
				`	"Alright, which worlds do you need me to land on?"`
				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
					decline

> 			`	They ask you to show them your map of the galaxy, so you bring them inside your ship, pointing to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`

Reviewer 8: ```suggestion
			`	They ask you to show them your map of the galaxy, so you bring them inside your ship and point out to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`
```

				accept
	npc "scan outfits"
		personality staying uninterested target
		government "Quarg"
		system "Enif"
		fleet
			names "quarg"
			variant
				"Quarg Skylark"
				"Quarg Wardragon"
	on complete
		"coalition jobs" ++
		payment 200000
		conversation
			`Once again the engineers remove your outfit scanner, taking a copy of the scan logs and handing it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`
			`	When the engineers return, they put your outfit scanner back in place, and the Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
			`	He wishes you safe travels, and leaves with his colleagues.`

> 

Reviewer 8: I'm just putting a comment here to remind me where I left off when I come back to this.

			`	They immediately set to repair your ship, and you're still on alert for a few hours in case any more ships come through, but everything appears to be normal. The station returns to normality as well: communications and other functions are restored.`
			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortup and inform him of what happened. Shortly after, <payment> are added to your account.`
			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain," he says.`
			`	He discusses with you the outfits that the ships were using and is equally alarmed that they had not only Heliarch equipment, but had also acquired human weapons.`
			`	"To the other consuls, this information I will bring. Further tasks, of you we may ask, but for now, done your work is, Captain," he ends the transmission after a short salute.`

event "resistance fleeing rb"
	system "12 Autumn Above"
		remove fleet "Heliarch"

event "resistance detained rb"
	system "12 Autumn Above"
		add fleet "Heliarch" 800

mission "Heliarch Drills 1"
	name "Refuelling Drill"

> 	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in desperate scenarios."

Reviewer 9: Suggested this change because the two different phrases mean the same thing, but "emergency situations" fits a little better when talking about the need to refuel instead of things sounding desperate. Purely subjective though. 

```suggestion
	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in emergency situations."
```

			choice
				`	"What do you mean?"`
					goto explain
				`	"Will I be fighting them?"`
			`	"No, no fighting," she says. "Not yet. Only simulating a pursuit, they will be."`
			label explain
			`	She points to the systems again, "To land on <planet> and Ashy Reach unseen by the Hunters, your task is. Attack you, they will not, but allow them to be in the system when you land, you cannot. Once cleared both planets, you have, return here for 300,000 credits, you may."`
			choice
				`	"Ok, I'll head out to the planets then."`
					goto accept
				`	"Can you tell me where my pursuers will be?"`
			`	"Know of their pursuers, the pursued do not," she says, letting a laugh as if it was a joke. "Well, usually... From many systems across the Coalition, the Hunters are. Ready to follow your ship when you launch, they will be."`
			label accept
			`	She points again to one of the planets, "Still preparing, some Hunters around Ashy Reach are, so head for <planet> first, you should," she wishes you luck on avoiding the Heliarch Hunters, and heads off to a Heliarch military building.`
				accept
	npc evade

> 		government "Heliarch"

Reviewer 3: I wish there was some way to tell these Hunters apart, like a slightly different color or smth like that, because rn I can't make them apart, and just ended up landing with one each time without realizing it, and then I randomly jump drived away to come back.

Author: I'll make them `target`.

	description "Head to the <destination>, where the Heliarch consuls will make you a Heliarch agent."
	minor
	landing
	source
		near "Quaru" 1 100
	destination "Ring of Friendship"
	to offer
		or
			"assisted heliarch" == 5
			and
				"coalition jobs" >= 90
				"assisted heliarch" == 4
			and
				"coalition jobs" >= 120
				"assisted heliarch" == 3
		not "Resistance: Questions: done"
	to fail
		has "Resistance: Questions: done"
	on offer
		conversation
			`When you land on <origin>, you find a Heliarch delegation waiting for you outside. When you meet them, the Saryd of the group says, "A great ally to our Coalition, you have proven to be."`
			`	"Discussed your exploits, the consuls have. Much contentment, your contributions brought us," the Kimek continues.`
			`	"With you, a meeting they request. To <destination> you must go, if to join our ranks, you so wish," the Arach finishes.`
				accept
	on complete
		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology, and a translation device, to freely converse with the Coalition species."
		set "language: Coalition"
		event "first heliarch unlock"
		set "license: Heliarch"
		conversation

> 			`When you land, you're met with a bit of ceremony: a trio of Heliarch consuls welcomes you, each carrying a gemstone, an emerald for the Saryd, a ruby for the Kimek, and a sapphire for the Arach. They each pass the precious stones to you, first the Saryd, then the Kimek and lastly the Arach. Cameras and photographers surround the scene, kept at bay by lines of Heliarch guards. It seems your joining the ranks has turned into something of an event.`

Reviewer 9: ```suggestion
			`When you land, you're met with a bit of ceremony. A trio of Heliarch consuls welcomes you, each carrying a gemstone: an emerald for the Saryd, a ruby for the Kimek, and a sapphire for the Arach. They each pass the precious stones to you, starting with the Saryd, then the Kimek, and lastly the Arach. Cameras and photographers surround the scene, kept at bay by lines of Heliarch guards. It seems your joining the ranks has turned into something of an event.`
```

			choice
				`	"Alright, what do you need me to do?"`
					goto accept
				`	"What would these 'more interesting drills' involve?"`
				`	"Sorry, I'm not interested in that kind of work."`
					decline
			`	"Thought of that, I still haven't," she says. "Some ship combat training, potentially. Very beneficial, training with an outsider could be."`
			choice
				`	"Alright, so should I just follow your ships now?"`
				`	"Sorry, but I'm not interested in that kind of work."`
					decline
			label accept
			`	"Follow you to <planet>, my ships will," she says, pointing at the Judicators. "In a rush to head back, we are not, so if immediate business elsewhere, you have, follow you for a while, we could. An extension to the refuelling drills, it would be."`
			`	She tells you she'll pay you <payment> when you reach <planet>, and will prepare another "drill" for you to help with too. She heads back inside one of the Judicators, and they prepare to follow your ship when you launch.`
				accept
	npc accompany save
		personality escort heroic opportunistic
		government "Heliarch"
		fleet
			names "heliarch"

> 			fighters "heliarch fighter"

Reviewer 10: ```suggestion
			fighters
				names "heliarch fighter"
```

	npc board
		personality timid plunders entering target
		government "Lunarium"
		ship "Kimek Thistle (Torpedoes)" "Resheph"
		conversation
			`This ship is equipped with Torpedo Launchers. Somehow, the criminals have acquired human outfits. You're about to enter the ship and look for any hostages that might have been there, but you rush back when you notice its self destruction sequence start.`
				launch
	on complete
		"reputation: Lunarium" = 1
		event "resistance detained rb"
		"assisted heliarch" ++
		payment 1320000
		conversation
			`Though you never managed to find out if there were any hostages on board, the Heliarchs stationed at <planet> still herald you as a hero for "detaining the bandits". They immediately set to repair your ship. You're still on alert for a few hours in case any more ships come through, but nothing appears to be out of the ordinary. Near the end of the day, the station returns to normal as well; communications and other functions are restored.`
			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortup and inform him of what happened. Shortly after, <payment> are added to your account, and you're put on a call with the consul.`

> 			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and, is equally alarmed that they had not only Heliarch equipment, but had also acquired human weapons. He lets out something you can't quite put, resembling a gasp or a roar; you only know that your ears don't want to hear it again.`

Reviewer 7: ```suggestion
			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and is alarmed to learn that they had not only Heliarch equipment, but had also acquired human weapons. He lets out something you can't quite put, resembling a gasp or a roar; you only know that your ears don't want to hear it again.`
```

		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Sol Kimek"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "3 Spring Rising"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "5 Summer Above"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "3 Pole"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "12 Autumn Above"
	on visit
		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`

mission "Heliarch Drills 2-C"
	landing
	name "Pursuit Drill"
	description "Land on the <planet> while avoiding the Heliarch Hunters."
	source "Ashy Reach"
	destination "Belug's Plunge"
	to offer
		has "Heliarch Drills 2-B: done"
		not "Resistance: Questions: done"
	on offer
		conversation

> 			`Once again, you managed to avoid the Heliarch Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me Captain. Forgotten something, I had. Heard of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`

Reviewer 7: ```suggestion
			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me Captain. Forgotten something, I had. Heard of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```

			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`
				accept
	npc disable save
		government "Heliarch Test Dummy"
		personality disables staying nemesis heroic
		system "Torbab"
		fleet
			names "heliarch"
			variant
				"Heliarch Punisher (Scrappy)"
	on fail
		"reputation: Heliarch" = -1000
		"reputation: Coalition" = -1000
	on complete
		"coalition jobs" ++
		"assisted heliarch" ++
		payment 800000
		conversation
			`Magister Lotuora greets you when you return. "Prevailed over one of our Punishers, you have! Perhaps the real threat, you are, not the Quarg," she says, looking at you for a few seconds as if expecting you to laugh. She hands you <payment>, and continues, "Sent a recording of the fight, I was. Learn much from it, I hope we can."`

> 			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She finally wishes you safe travels, heading to a Heliarch building.`

Reviewer 7: ```suggestion
			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She finally wishes you safe travels, then heads to a Heliarch building.`
```

		conversation
			action
				clear "resistance evidence"
			`You receive a message from the Heliarch consul Iekie as you go in for a landing. "Captain <last>! Analyzed your findings, we have. A great boon, your assistance was, having located two stashes of our stolen equipment. Investigating who was involved in such operations, my teams are. Very grateful for your help, we are, so sending a monetary bonus as thanks, I am."`
			`	When she finishes, <payment> are transferred to your account.`
				decline

mission "HI2-Bonus 3"
	invisible
	landing
	to offer
		has "Heliarch Investigation 2: done"
		"resistance evidence" == 3
	source
		government "Coalition"
	on offer
		payment 1200000
		conversation
			action
				clear "resistance evidence"
			`You receive a message from the Heliarch Consul Iekie as you go in for a landing. "Captain <last>! Analyzed your findings, we have. Worry me greatly, all these stashes of stolen equipment do. Indebted to you, we are, as without your help found all these operations, we would not have. A monetary bonus, as thanks, I am sending. Deserve it, you do, Captain."`
			`	When she finishes, <payment> are transferred to your account.`
				decline

> mission "HI2-MP"

Author: Updated them.

			`	At first glance there is nothing inside, but after he furiously crawls all over the place, he puts his head to a large metal panel on the opposite end of the room, before shooting that too. You approach him to find a hidden staircase leading underground.`
			`	On your way down, Kirrie yells for you to take cover, and you barely escape a barrage of shots as you head back up. Kirrie isn't as lucky, as one of the shots hit one of his back legs just before he is back at the stairs. You go to help him up before any more shots come, but he seems to make do with just five good legs well enough.`
			`	You two head back up, and Kirrie tells you to be alert and watch the staircase, as he reaches for a small radio-like device to call for support. A few minutes later, a pair of Heliarch Punishers land nearby, and dozens of armed agents enter the warehouse, while you head to one of the ships with Kirrie, and wait as his wound is treated.`
			`	The agents return hours later, reporting they shot down the turrets and found the place deserted. "Many automated production lines, we found," one of them tells Kirrie. "Powering them, one of our Reactor Modules is."`

> 			`	With the local Heliarchs taking care of the place, they have another, smaller ship come in to pick you two up, and drop you and Kirrie closer to the <ship>, where you find Iplora and Tugroob already waiting. You all head inside, and Kirrie suggests you depart now, as his colleagues follow him to his bunk to hear what happened.`

Reviewer 11: It looks like the Punishers do.

mission "HI2-SS"
	name "Investigate <planet>"
	description "Transport the Heliarch trio to <destination> so that they can investigate signs of criminal activity."
	to offer
		has "Heliarch Investigation 1: done"
	source "Ring of Friendship"
	destination "Shifting Sand"
	to fail
		or
			has "Heliarch Investigation 2: declined"
			has "Heliarch Investigation 2: aborted"
			has "Heliarch Investigation 2: failed"
	on complete
		conversation
			`Iplora leaves the ship immediately, disappearing into the spaceport. Tugroob and Kirrie say that she is heading to a nearby oasis.`
			`	"Like to connect with nature, the Saryds do. Hoping to hear rumors from the others there, she is," Kirrie explains as he also prepares to leave, claiming to know a few Heliarchs stationed here who could help him look into some reports of a local smuggler's base.`
			`	Being the last Heliarch agent in your ship, Tugroob turns to you and asks, "Assist me in my search, would you, Captain?"`
			choice
				`	"Of course. Where are we headed?"`
				`	"Sorry, I'll have to refuse this time."`
					goto decline

> 			`	"An abandoned mining operation our target is. Received we have reports of suspicious activity near the complex, ships that recently there headed in spite of it shutting down years ago," he explains, and you input the coordinates he gives you into your navigator.`

Author: Edited it some more, if you could check if it's alright now.

			`	The place is surprisingly tidy, if dusty near the door with what sand's been let and the previous owners didn't bother cleaning out before packing up and leaving. Various cabinets, archives and work stations dominate the initial room, and you can see it connects to a collective bedroom, filled with bunks, and a storage area.`
			`	"No other entrances there seem to be. If hiding anything here, others were, force their way in they had to, just like us," Tugroob says.`
			`	Given the storm's trapped you in, you two turn over every drawer, archive and box, look over everything in the storage area, try and get the computers to work, to no avail, and even check the bedroom, and its connecting dining area and restrooms. Your search doesn't bear fruit, though, and so you're left waiting inside as the storm rages around the building.`
				goto fail
			label tunnel
			action
				"resistance evidence" ++

> 			`	You and Tugroob start checking the vehicles first, then briefly look into the tunnels they dug into the sides of the mining hole. The holes mostly head in straight lines down, forming straightforward paths to where the lesser ore veins were. Knowing how the vehicles operate better than you, Tugroob checks them first, before joining you in exploring the tunnels.`

Reviewer 11: ```suggestion
			`	Having more skill with the vehicles, Tugroob checks them first, before joining you in exploring the tunnels they dug into the sides of the mining hole. The holes mostly head in straight lines down, forming straightforward paths to where the lesser ore veins were.`
```

			`	With your focus on blocking a gust of snow, you end up tripping over another one of the mounds on your way, and realize you don't remember which cave Iplora entered. Considering the weather, though, taking shelter in any of them would be a good choice.`
			choice
				`	(Enter the nearest cavern.)`
					goto shelter
				`	(Try and dig through the snowy mounds to find something.)`
					goto dig
			label cave
			`	You and Iplora bring out your flashlights as you two explore the caverns. After some twists and turns she notices some large rocks that evidently broke free from the walls, but still hug them closely. She puts down her flashlight, and starts pushing the rocks, revealing a passage to a small campsite containing some makeshift beds, knocked over equipment boxes, and a worn down campfire in the middle.`

> 			`	You search through the boxes as she examines the room looking for some other hidden area, but neither of you find anything. She checks the campfire and concludes that nobody has been here for a few months. You continue exploring the cave and eventually loop back outside, leaving via another cave entrance, this time much further from the forest according to Iplora. Given the blizzard is now strong enough that you've lost sight of the forest from earlier, you wonder how she's even managing to tell where you are, and for how much longer she'll be able to do so.`

Reviewer 7: ```suggestion
			`	You search through the boxes as she examines the room looking for some other hidden area, but neither of you find anything. She checks the campfire and concludes that nobody has been here for a few months. You continue exploring the cave and eventually loop back outside, leaving via another cave entrance that, according to Iplora, is this time much further from the forest. Given the blizzard is now strong enough that you've lost sight of the forest from earlier, you wonder how she's even managing to tell where you are, and for how much longer she'll be able to do so.`
```

				accept
			label snowdig
			`	You start clawing your way down the snow as Iplora asks you to stop and continue moving toward the ship. You continue, your hands getting colder as the snow steals the heat even through your gloves. You hit a hard surface and start clearing from there, only to see it was, in fact, just a boulder.`
			`	"Satisfied? Return, we must. Come now," You turn to see her offering her hand once again.`
			choice
				`	(Take her hand.)`
					goto back
				`	(Head to another mound and try digging there.)`
			label keepdig
			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow muffling the sounds around you, you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, this time creating a faint metallic echo. She takes her hands off of you and lays down, now helping you dig out whatever is there. Once the object is visible, she stands up. Though much of it is still buried, you can tell it's a very large metal container of sorts - the part that's out of the snow looks almost like the larger end of a coffin.`

> 			`	After clearing out what snow you could from over the rest of the object, you see it's many times larger than either of you. You try to look for some way to open it while she starts going over her bag, looking for something, but the cold, frost-plagued metal doesn't budge to your efforts. She finally seems to find what she was looking for, some sort of comms device, but before she has a chance to activate it, an energy blast from the far side of the resort booms the ground nearby. It misses her by only a meter or so, and melts much of the snow it hit to nothing.`

Reviewer 7: ```suggestion
			`	After clearing out what snow you could from over the rest of the object, you see it's many times larger than either of you. You try to look for some way to open it while she starts going over her bag, looking for something, but the cold, frost-plagued metal doesn't budge to your efforts. She finally seems to find what she was looking for, some sort of comms device, but before she has a chance to activate it, an energy blast from the far side of the resort rattles the ground nearby. It misses her by only a meter or so, and melts much of the snow it hit to nothing.`
```

> 			`	She yells for you to get down, pulling out a Heliarch blaster weapon from the bag, and firing back in the direction the shot came from.`

Reviewer 7: ```suggestion
			`	She yells for you to get down, pulling out a Heliarch blaster weapon from the bag and firing back in the direction the shot came from.`
```

			action
				"resistance evidence" ++
			choice
				`	(Take cover.)`
				`	(Pull out your gun and help her.)`
					goto help
			`	You rush to the nearest resort house, staying behind it and hoping you'll have enough protection from the blasts. You can still hear Iplora move around, and you occasionally peek your head over to see her shooting and taking cover behind the houses herself, running to them and narrowly avoiding incoming blasts as she tries to close in the distance with whoever or whatever is shooting. The shots stop suddenly, and you see her come over, saying that it should be safe to get out now.`
			`	"Fled, our attacker has," she claims. From the looks of it she wasn't injured. You two get back to the dug out spot, and Iplora grabs her bag. She managed to call for support in between running for cover and shooting back, and you two wait by one of the houses for a few minutes.`
				goto opencrate
			label help

> 			`	You two start shooting back as you move for cover behind the resort houses. Iplora signals for you to head out to another house, while she takes a different path. Rushing to the next house for cover while shooting at where the shots are coming from, you narrowly avoid the ones that come your way. You lose sight of Iplora herself, now only making out the bright blasts from her weapon, as she moves quickly through the resort.`

Reviewer 7: ```suggestion
			`	You two start shooting back as you move for cover behind the resort houses. Iplora signals for you to head out to another house, while she takes a different path. Rushing to the next house for cover while shooting in the direction of your attacker, you narrowly avoid the shots that come your way. You lose sight of Iplora herself, now only making out the bright blasts from her weapon as she moves quickly through the resort.`
```

			`	Trying to keep up, you notice the shots grow less frequent as they alternate between you and Iplora. The blizzard is bad enough that you can't even see the attacker despite getting closer and closer, but it seems that goes both ways, as you and Iplora manage to avoid getting hit without too much trouble.`

> 			`	You've barely gotten past half of the resort when the shots stop altogether. Iplora still takes aim for a bit, firing some shots as she continues on past some houses, but ultimately lowers her weapon, coming toward you. "Fled, our attacker has," she claims. You two head back to the dug out spot, and Iplora grabs her bag. She managed to call for support in between running for cover and shooting back, so you two wait by one of the houses for a few minutes.`

Reviewer 7: ```suggestion
			`	You've barely gotten past half of the resort when the shots stop altogether. Iplora still takes aim for a bit, firing some shots as she continues on past some houses, but ultimately lowers her weapon and comes toward you. "Fled, our attacker has," she claims. You two head back to the dug out spot, and Iplora grabs her bag. She managed to call for support in between running for cover and shooting back, so you two wait by one of the houses for a few minutes.`
```

			label opencrate
			`	Several Heliarch Neutralizers arrive just before the blizzard subsides, and you're directed to one of them to dry your clothes and make sure you're not frostbitten. Nearly all of the Heliarch agents making the rounds over the resort are Saryds, and are quick to locate more of the coffin-like containers buried in the snow. They pull them out with equipment they've brought in the ships, and carefully melt down the ice and snow layers covering them, finally managing to force them open once they've dried out.`

> 			`	Each one houses a Heliarch Finisher Torpedo inside, to the agents' surprise. As they open them up and reveal dozens of the torpedoes, Iplora heads out with a group of agents, showing them signs of the shootout with the attacker, and giving her thoughts on where they could be fleeing to. She also instructs another group on the local caverns, before they head underground. When she's back with you, one of the captains of the Neutralizers tell you that they will handle everything from here, and have one of the ships take you two to where your own ship landed, leaving as you prepare to take off.`

Reviewer 7: ```suggestion
			`	Each one houses a Heliarch Finisher Torpedo inside, to the agents' surprise. As they open them up and reveal dozens of the torpedoes, Iplora heads out with a group of agents, showing them signs of the shootout with the attacker, and giving her thoughts on where they could be fleeing to. She also informs another group on the layout of the local caverns before they head underground. When she's back with you, one of the captains of the Neutralizers tell you that they will handle everything from here, and has one of the ships take you two to where your own ship is parked. He leaves as you prepare to take off.`
```

				accept
	npc "scan outfits"
		personality staying uninterested target
		government "Quarg"
		system "Enif"
		fleet
			names "quarg"
			variant
				"Quarg Skylark"
				"Quarg Wardragon"
	on complete
		"coalition jobs" ++
		payment 200000
		conversation
			`Once again the engineers remove your outfit scanner, taking a copy of the scan logs and handing it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`
			`	When the engineers return, they put your outfit scanner back in place. The Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
			`	He wishes you safe travels, and leaves with his colleagues.`

mission "Quarg Interrogation"
	invisible
	landing
	deadline 1
	source
		government "Quarg"
		not planet "Humanika"
	destination "Earth"
	to offer
		has "Heliarch Recon 2-B: done"
	on offer
		conversation

> 			`As you land on <origin>, a Quarg is patiently waiting right outside. Seeing that it won't go away, you head outside to speak with it.`

Reviewer 11: I think this could be expanded. Are you just sitting inside, waiting for them to leave, or do you try something? Does the Quarg make some gesture for you to come out, or that they won't leave?

		`	Once again you're led through the inscrutable, guard-crowded halls of the <origin>, being brought to a long corridor with dozens of closed doors, with the one at the very end apparently being your goal. From there, however, two more guards are coming, and they explain to those escorting you that "received word, the Ring of Friendship has." They apologize to you, saying Aulori might be busy for some time more, but bring you to just outside the room, so that you're received as soon as his "meeting" ends.`
		`	From another room, a Heliarch arbiter waves for the guards waiting with you, and passes some other task to them. Three of them leave, and you're left with one of those who fetched you from your ship, a Saryd. Despite the door to Aulori's room being locked, you two can still clearly hear intense shouting from within - not just his, but from many others who you assume to be consuls as well. After a particularly long - and heated - debate segment from what seems to be Aulori's voice, the Heliarch guard waiting with you declares they'll be coming back soon, and hastily move away, turning the corner.`
		branch license

> 				has "Heliarch License: done"

Reviewer 12: ```suggestion
			has "Heliarch License: done"
```

					decline
			label accept
			`	They thank you for your cooperation, and call in a device for the rest of the agents to come to your ship, along with their cargo. After securing the last of the containers in your hold, you show them to their bunks, and head to the cockpit.`
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on complete
		payment 420000
		dialog `You're told to open your hatch as soon as you land, and when you do, the Heliarch agents storm out of your ship while a separate group that was waiting for you unloads the cargo. A few seconds later you see that <payment> have been added to your account.`

mission "Heliarch Containment 2"
	name "Pick Up Injured Soldiers"
	description "Land on <stopovers>, to pick up the <bunks> Heliarch soldiers in need of medical attention, and transport them to <destination> by <date>."
	deadline
	passengers 21
	to offer
		has "Heliarch Containment 1: done"
		random > 35
	source
		near "Bloptab" 1 3
	stopover "Delve of Bloptab"
	destination "Ahr"
	on offer
		conversation
			`You're stopped by a group of Heliarch agents as you enter the spaceport, most of them Arachi.`

> 			`	"Injured, many colleagues of ours have been. On <stopovers>, they are," one of them says. "Dealing with containing criminal operations, they were, surprised them with an attack amidst civilians, the rogues did. Given the circumstances, dangerous for them, it might be, if treated within hospitals there, they are."`

Reviewer 7: ```suggestion
			`	"Injured, many colleagues of ours have been. On <stopovers>, they are," one of them says. "Dealing with containing criminal operations, they were, and surprised them with an attack amidst civilians, the rogues did. Given the circumstances, dangerous for them, it might be, if treated within hospitals there, they are."`
```

	name "Undercover: Payment"
	description "Collect your payment for escorting the Heliarch transports."
	landing
	to offer
		has "Heliarch Containment 4-A: done"
		not "Resistance: Questions: done"
	source "Remote Blue"
	destination "Station Cian"
	on offer
		conversation
			`Hundreds of Heliarch agents pour out of the three Kimek transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand. From the reactions of the locals, it's clear they were neither expecting such a thing, nor are they happy with the agents going about the city in that manner.`
			`	When the last of the Heliarchs leave the transports, one of the Kimek captains inform you that you should now head to <destination> to collect your payment of <payment>, and to "avoid getting caught in any potential retaliation."`
				accept
	on complete
		payment 495000
		conversation
			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`

> 			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but trouble you with that, I will not. Very useful to us your support has been, Captain."`

Reviewer 7: ```suggestion
			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Very useful to us your support has been, Captain."`
```

mission "Heliarch Containment 5"
	name "Stop The Fleeing Ships"
	description "Defeat the criminal ships that are attempting to flee. You must disable and board their ships, in order to help the Heliarchs find the source of their weapons."
	landing
	to offer
		has "Heliarch Containment 4-B: done"
		not "Resistance: Questions: done"
	source "Station Cian"
	on offer
		"reputation: Lunarium" = -1000
		event "resistance fleeing rb"
		conversation

> 			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything particularly interesting, as the station holds few shops. You move to the job board, but find it completely empty. You also notice that the market's trade values are not updating, and any news system is non-functional.`

Reviewer 7: ```suggestion
			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything particularly eye-catching, as the station holds few shops. You move to the job board, but find it completely empty. You also notice that the market's trade values are not updating, and any news system is non-functional.`
```

			choice
				`	"What's going on?"`
				`	"My ship isn't exactly combat-ready to fight your own warships."`
			`	They explain that it's not simply the station's mercantile and news systems that haven't been functioning properly - they haven't been able to contact the group on Remote Blue, nor the fleet that left for Fourth Shadow, nor get in any sort of contact with anywhere else in the Coalition.`
			`	"Only readings of the local systems, we can get, and alerted us, they have, of incoming ships! Armed, civilian ships! Criminals, no doubt, and fleeing Remote Blue, they are!"`
			`	You can't say much before they've practically pushed you back into your ship. They promise you'll be greatly compensated for this, and ask that you only disable the ships so you can board and check for hostages, or possibly allow for capture and questioning of the outlaws.`
				launch
	npc board
		personality timid plunders entering target
		government "Lunarium"
		ship "Saryd Sojourner (Bombardments)" "Hodoviah"
		conversation

> 			`Unlike the other two vessels, this ship is equipped with Heliarch weaponry. Unfortunately you can neither verify the equipment it is using nor check for hostages, as a self destruction sequence starts when you're about to enter it.`

Reviewer 4: The player might not know what the other two ships are equipped with.

Author: Changed it to not expect the other two were boarded first.

	npc board
		personality timid plunders entering target
		government "Lunarium"
		ship "Kimek Thistle (Torpedoes)" "Resheph"
		conversation
			`This ship is equipped with Torpedo Launchers. Somehow, the criminals have acquired human outfits. You're about to enter the ship and look for any hostages that might have been there, but you rush back when you notice its self destruction sequence start.`
				launch
	on complete
		"reputation: Lunarium" = 1
		event "resistance detained rb"
		"assisted heliarch" ++
		payment 1320000
		conversation
			`Though you never managed to find out if there were any hostages on board, the Heliarchs stationed at <planet> still herald you as a hero for "detaining the bandits". They immediately set to repair your ship. You're still on alert for a few hours in case any more ships come through, but nothing appears to be out of the ordinary. Near the end of the day, the station returns to normal as well; communications and other functions are restored.`
			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortup and inform him of what happened. Shortly after, <payment> are added to your account, and you're put on a call with the consul.`

> 			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and, is equally alarmed that they had not only Heliarch equipment, but had also acquired human weapons, letting out what you can't quite put to be a gasp or a roar, only knowing your ears don't want to hear it again.`

Reviewer 7: ```suggestion
			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and, is equally alarmed that they had not only Heliarch equipment, but had also acquired human weapons. He lets out something you can't quite put, resembling a gasp or a roar; you only know that your ears don't want to hear it again.`
```

event "resistance fleeing rb"
	system "12 Autumn Above"
		remove fleet "Heliarch"

event "resistance detained rb"
	system "12 Autumn Above"
		add fleet "Heliarch" 800

mission "Heliarch Drills 1"
	name "Refuelling Drill"
	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in emergency situations."
	minor
	source
		near "Belug" 6 100
	destination "Belug's Plunge"
	to offer
		has "Coalition: Contributor: done"
		not "Resistance: Questions: done"
		random < 8
		"combat rating" >= 8104
	on offer
		conversation
			`A pair of Heliarch Judicators land by your ship as you leave for the spaceport. The workers storm to the ships, getting ready to refuel them. Coming down from one of the Judicators, a young Saryd with black hair barely reaching her shoulders pulls out a watch of sorts, seeming to time how long they take to completely fill up the ships' fuel tanks.`

> 			`	When they're done, she gives an approving nod, and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refuelling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part, it is, in testing how well prepared for emergencies we are."`

Reviewer 7: ```suggestion
			`	When they're done, she gives an approving nod, and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refuelling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part it is, in testing how well prepared for emergencies we are."`
```

mission "Heliarch Drills 2-A"
	name "Pursuit Drill"
	description "Land on the <planet> while avoiding the Heliarch Hunters."
	source "Belug's Plunge"
	destination "Shadow of Leaves"
	to offer
		has "Heliarch Drills 1: done"
		not "Resistance: Questions: done"
	on offer
		conversation
			`When you find Magister Lotuora in the spaceport, she shows you to a map of the Coalition. Pointing to two systems, she indicates one in Kimek and one in Saryd space. "Made many calls, I have, and to assemble many ships for this drill, I've managed. How fit for their job, our Hunters are, we shall see."`
			choice
				`	"What do you mean?"`
					goto explain
				`	"Will I be fighting them?"`
			`	"No, no fighting," she says. "Not yet. Only simulating a pursuit, they will be."`
			label explain
			`	She points to the systems again, "To land on <planet> and Ashy Reach unseen by the Hunters, your task is. Attack you, they will not, but allow them to be in the system when you land, you cannot. Once cleared both planets, you have, return here for 300,000 credits, you may."`
			choice

> 				`	"Ok, I'll head out to the planets then."`

Reviewer 7: ```suggestion
				`	"Okay, I'll head out to the planets then."`
```

				accept
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Silver Bell"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Hunter"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Fallen Leaf"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Companion"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Good Omen"
	on visit
		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`

mission "Heliarch Drills 2-B"
	landing
	name "Pursuit Drill"

> 	description "Land on the <planet> while avoiding the Heliarch Hunters."

Reviewer 7: ```suggestion
	description "Land on <planet> while avoiding the Heliarch Hunters."
```

				accept
	on complete
		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology. Was also given a translation device, to freely converse with the Coalition species."
		set "language: Coalition"
		event "first heliarch unlock"
		set "license: Heliarch"
		conversation
			`When you land, you're met with a bit of ceremony. A trio of Heliarch consuls welcomes you, each carrying a gemstone: an emerald for the Saryd, a ruby for the Kimek, and a sapphire for the Arach. They each pass the precious stones to you, starting with the Saryd, then the Kimek, and lastly the Arach. Cameras and photographers surround the scene, kept at bay by lines of Heliarch guards. It seems your joining the ranks has turned into something of an event.`
			`	When the consuls are done, they have you follow them into the restricted sections of the ring, bringing you to a very large, circular room, somewhat like a colosseum or an auditorium, with hundreds of Heliarch consuls seated. There are also many more camera workers and photographers, though all of them are Heliarch members this time. You're instructed to head to the center.`

> 			`	The consuls who brought you here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much, this outsider has, and shown devotion to our Coalition, this outsider has."`

Reviewer 7: ```suggestion
			`	The consuls who brought you here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much and shown devotion to our Coalition, this outsider has."`
```

			`	"From humanity, <first> <last> has come," the Kimek continues. "Our most recent neighbor, and a group still under the chiding eyes of the Quarg, they are. To reach them all, to free them all, we still cannot, but, our first step in that front, Captain <last>'s coronation is."`
			`	The Saryd approaches you, carrying some type of chest. Opening it, she reveals a golden circlet inside, staying silent and looking at you.`
			choice
				`	"Do I put it on myself?"`
				`	(Take the circlet.)`
					goto circlet
			`	"Allow others to crown us, we do not," she says. "Not anymore."`
			choice
				`	(Take the circlet.)`
			label circlet

> 			`	You pick up the circlet, and raise it to your head, the Heliarch consuls all standing up when you do. You bring it down and adjust it to your head. Once the "crowning" is done, many of the consuls cheer in celebration, with something akin to a round of applause, and more of the speeches from before.`

Reviewer 7: ```suggestion
			`	You pick up the circlet and raise it to your head, the Heliarch consuls all standing up when you do. You bring it down and adjust it so that it fits you well. Once the "crowning" is done, many of the consuls cheer in celebration, with something akin to a round of applause, and more of the speeches from before.`
```

			choice
				`	"To my will? What do you mean?"`
				`	"Is this really necessary?"`
			`	"Mere jewelry pieces or symbols of rank these are not," the one tinkering with your circlet explains. "Serve they do as links, interfaces between a Heliarch's will and our Coalition's machines and programs. Allow you they will to access some of our restricted ring sections, and serve as well as keys to our many outfitters, they do."`
			`	They gesture for you to sit, and you comply, having the helmet placed over your head. It blocks out your sight completely, and feels a lot heavier that you had imagined. "Begin now, we shall," you barely hear one of the Heliarchs say as most of the sound is muffled. Once the process starts, you hear some wave-like humming echo inside your head, and it doesn't take long before you feel as if you're being spun around. Before you get too uncomfortable, the Heliarchs stop the machine, and take the helmet off of you. "All done we are. Here," they say handing you your circlet back.`

> 			`	Putting it on again, you don't notice any difference, but they push you to "try it out" on one of the doors. You get up from your seat, moving to the entrance door, and it opens up when you're a few steps away still. You think to close it, and it does. You repeat a few more times, and turn back to the Heliarchs, who seem amused by seeing you play with your new ability. You look now to the helmet, and with a thought it turns out, seemingly spinning around a head it assumes it is attached to. You turn it off, and test with the lights, and some of the computers there too.`

Reviewer 7: ```suggestion
			`	Putting it on again, you don't notice any difference, but they push you to "try it out" on one of the doors. You get up from your seat, moving to the entrance door, and it opens up when you're a few steps away still. You think to close it, and it does. You repeat a few more times, and turn back to the Heliarchs, who seem amused by seeing you play with your new ability. You look now to the helmet, and with a thought it turns around, seemingly spinning around a head it assumes it is attached to. You turn it off, and begin to test the circlet with the lights and computers in the room.`
```

> 			`	When you're done, the two Heliarchs explain some more, "As of a lower rank you are, permit you not access to everything and everywhere, your circlet will. Not yet, at least. But, as mentioned, come to some of the restricted sections, you now many. On many of our worlds, find you will Heliarch outfitters, where now available some of our equipment will be. Permit you to take on some jobs we put up for Heliarch agents here in the restricted sections, the circlet also will."`

Reviewer 7: ```suggestion
			`	When you're done, the two Heliarchs explain some more, "As of a lower rank you are, permit you access to everything and everywhere, your circlet will not. Not yet, at least. But, as mentioned, come to some of the restricted sections, you now many. On many of our worlds, find you will Heliarch outfitters, where now available some of our equipment will be. Permit you to take on some jobs we put up for Heliarch agents here in the restricted sections, the circlet also will."`
```

			`	The place is surprisingly tidy, if dusty near the door with what sand has been let in. Despite the relative cleanliness, it's evident that the previous owners didn't bother cleaning out before packing up and leaving. Various cabinets, archives and work stations dominate the initial room, and you can see it connects to a collective bedroom, filled with bunks and a storage area.`
			`	"No other entrances there seem to be. If hiding anything here, others were, force their way in they had to, just like us," Tugroob says.`
			choice
				`	"Doesn't look like anyone's been in here in a while. Let's check the mining hole."`
					goto hole2
				`	"There might be something hidden here. Let's look around."`
			`	You two go about checking every drawer, archive and boxes of all sizes in the first room, but only end up finding a few scattered documents related to the mine, and some old doodles made by what must have been very bored workers. Tugroob tries to get the computers on, but no manner of tinkering brings any electricity to them. The building really is completely out of power.`

> 			`	"Find nothing here we will, Captain," Tugroob says, looking to the outside at the approaching sandstorm.`

Reviewer 13: ```suggestion
			`	"Find nothing here we will, Captain," Tugroob says, looking out the door at the approaching sandstorm.`
```

			`	"Request support equipment to get through safely, we must," he says, and using a communication device, he calls for assistance from Heliarch agents, warning them about the sandstorm. You head back up to the building to wait for them.`
			`	Shortly after, several dozen Heliarch agents arrive. Most of them work to keep the sand out, while a few follow you two down and inside the peculiar tunnel, bringing some form of cutting tool and several pieces of sensor equipment. After checking with the sensors for a few minutes, they use the tool to surgically remove what Tugroob was slowly examining before: several extremely thin wires, connected to some type of machinery hidden within the wooden beams.`

> 			`	After they're done, the agents storm the place but find nobody. What they do find are dozens upon dozens of crates filled to the brim with Heliarch staves, blaster guns, riot gear and other weaponry.`

Reviewer 13: ```suggestion
			`	After they're done, the agents storm the place but find nobody. What they do find are dozens upon dozens of crates filled to the brim with Heliarch staves, blaster guns, riot gear, and other weaponry.`
```

			choice
				`	(Go inside with Iplora.)`
					goto cave
				`	(Search around the snow field.)`
			`	You tell her you'll look around the snow for a while and go inside if the storm gets rough again. Walking around in the developing blizzard, this time minding the bumpy ground below, you try to look for anything off about any of the resort homes, or perhaps spot something in the snow itself. Although the scenery is a great sight, the task quickly becomes dull. Even after walking around the entire resort, your efforts produce no results. The snow starts falling much faster, so you begin making your way back to the caverns.`
			`	With your focus on blocking a gust of snow, you end up tripping over another one of the mounds on your way, and realize you don't remember which cave Iplora entered. Considering the weather, though, taking shelter in any of them would be a good choice.`
			choice
				`	(Enter the nearest cavern.)`
					goto shelter
				`	(Try and dig through the snowy mounds to find something.)`
					goto dig
			label cave

> 			`	You and Iplora bring out your flashlights as you two explore the caverns. After some twists and turns she notices some large rocks that evidently broke free from the walls, but still hug them closely. She puts down her flashlight and starts pushing the rocks, revealing a passage to a small campsite containing some makeshift beds, knocked over equipment boxes, and a worn down campfire in the middle.`

Reviewer 13: ```suggestion
			`	You and Iplora bring out your flashlights as you two explore the caverns. After some twists and turns, she notices some large rocks that evidently broke free from the walls, but still hug them closely. She puts down her flashlight and starts pushing the rocks, revealing a passage to a small campsite containing some makeshift beds, knocked over equipment boxes, and a worn down campfire in the middle.`
```

	to offer
		has "Heliarch Investigation 1: done"
	source "Ring of Friendship"
	destination "Remote Blue"
	to fail
		or
			has "Heliarch Investigation 2: declined"
			has "Heliarch Investigation 2: aborted"
			has "Heliarch Investigation 2: failed"
	on complete
		conversation
			`When you land, a couple of other civilian ships land nearby. Some Kimek leave them and wait outside your own. Kirrie is the only one to leave, and after speaking a bit with the other Kimek, they leave for the city, splitting in two groups.`
			`	"Not fond of outsiders, the people of this world are," Tugroob explains. "Decided to employ undercover Kimek agents, Consul Iekie has."`
			`	You wait around for a couple hours as Iplora and Tugroob tinker with your ship's communication devices, keeping in contact with the two groups. They don't mention if either group finds anything, and as night falls, Kirrie returns to the ship on his own. He doesn't say anything of the operation, instead just heading to his bunk to rest. Iplora and Tugroob follow suit, after telling you to depart whenever you're ready.`

mission "Heliarch Recon 1"
	name "Heliarch Reconnaissance"

> 	description "Fly to all the Quarg planets and stations the Heliarch asked, land on them, let your modified ship sensors scan their surroundings, then head back to the <destination>."

Reviewer 13: ```suggestion
	description "Fly to all the Quarg planets and stations the Heliarch asked, land on them, let your modified ship sensors scan their surroundings, and then head back to the <destination>."
```

	minor
	source
		government "Heliarch"
	stopover "Lagrange"
	stopover "Forpelog"
	stopover "Grakhord"
	stopover "Alta Hai"
	stopover "Kuwaru Efreti"
	destination "Ring of Wisdom"
	to offer
		"reputation: Quarg" >= 0
		"coalition jobs" >= 55
		has "visited planet: Lagrange"
		has "visited planet: Alta Hai"
		has "visited planet: Kuwaru Efreti"
		has "First Contact: Hai: offered"
		has "Coalition: Contributor: done"
		not "Lunarium: Questions: done"
	on offer
		require "Jump Drive"
		conversation
			`Three Heliarch agents are waiting for you when you land. "The human visitor who so much to our society has contributed, we greet," the Saryd says as they all salute you.`
			`	"Sent by Consul Aulori, we were," the Kimek says proudly, as if you were supposed to know who that is. "An important task for you, he has in mind."`
			`	"How far would the visitor go, if to further help our Coalition, it meant?" the Arach asks.`
			choice
				`	"I'd be glad to help. What do you need?"`
				`	"I'm not sure I understand what you're getting at."`

> 				`	"Obviously I have my limits as for what I would do."`

Reviewer 13: ```suggestion
				`	"Obviously, I have my limits."`
```

			`	"A jump drive, you possess. Far away, you can travel. Bother you, the harsh cages of the Quarg do not," the Saryd explains as all three walk toward your ship.`
			`	"Few in number, our own jump drives are, and too risky would it be, for one of our own to venture far out," the Kimek adds.`

> 			`	"If to help us gain knowledge of the opressors, you wish, lend the Coalition your ship's sensors for a short time, would you?" asks the Arach.`

Reviewer 13: ```suggestion
			`	"If to help us gain knowledge of the oppressors, you wish, lend the Coalition your ship's sensors for a short time, would you?" asks the Arach.`
```

			`	"Land on the rings, you really did, right Captain?" the Saryd asks while pointing at the places you visited.`
			`	You nod, and the Kimek hits your control panel lightly with one of his legs. "No data on the rings, the sensors collected! Gone wrong, something must have."`
			`	They have engineers return your sensors to normal, while you explain how your ship behaved while landing on them. They speak among themselves for a while, then the Arach says, "Too straightforward an approach, we might have taken. Captain <last>, contact you again soon, we will."`
			`	They hand you <payment>, and leave to one of the Heliarch sections of the ring.`

mission "Heliarch Recon Alta Hai"
	invisible
	landing
	source "Alta Hai"
	destination "Earth"
	to offer
		has "Heliarch Recon 1: active"
	on offer
		conversation

> 			`Coming in for a landing, something from inside your ship produces an irritating, high pitched whistle sound. It grows stronger the closer you get to the ringworld, until your ship completely shuts down. Your controls are rendered useless as you attempt to land manually, only for your autopilot to click back to normal and land as usual at the last moment. Looking outside, you spot several Quarg standing still, staring at your ship, until they hastily leave without contacting you at all.`

Reviewer 13: ```suggestion
			`Coming in for a landing, something from inside your ship produces an irritating, high pitched whistle sound. It grows stronger the closer you get to the ringworld, until your ship completely shuts down. Your controls are rendered useless as you scramble to land manually, only for your autopilot to click back to normal and land as usual at the last moment. Looking outside, you spot several Quarg standing still, staring at your ship, until they hastily leave without contacting you at all.`
```
Personal preference. The shutdown seems sudden and `<first>` doesn't have to manually land often, so I feel this captures the situation better.

				accept
	npc "scan outfits"
		personality staying uninterested target
		government "Quarg"
		system "Enif"
		fleet
			names "quarg"
			variant
				"Quarg Skylark"
				"Quarg Wardragon"
	on complete
		"coalition jobs" ++
		payment 200000
		conversation
			`Once again the engineers remove your outfit scanner, take a copy of the scan logs, and hand it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`
			`	When the engineers return, they put your outfit scanner back in place. The Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
			`	He wishes you safe travels, and leaves with his colleagues.`

mission "Quarg Interrogation"
	invisible
	landing
	deadline 1
	source
		government "Quarg"
		not planet "Humanika"

> 	destination "Earth"

Author: It is `deadline 1` so it should be out of the save file upon next landing already I think?

> 	destination "Earth"

Reviewer 13: Oops lol, you're right my bad

				`	"Sorry, I don't like the sound of this."`
			label decline
			branch license3
				has "Heliarch License: done"
			`	They are both surprised and distraught by your refusal, clearly not having expected it. They insist a bit further, but eventually give up on trying to convince you to help. Disappointed, they leave for a Heliarch office.`
				decline
			label license3
			`	They are both surprised and distraught by your refusal, clearly not having expected it. Their expressions then coalesce into cold, fearful gazes. They whisper briefly among themselves, then the Saryd addresses you.`
			`	"Captain, insist we must. An order from the consul, it is. In our ranks now you are, so do you should as dictates the chain of command."`
			`	"Lose your rank, you could, if to disobey such an order you were. For the best it is that continue aiding us and the consul, you do," the Kimek continues.`
			`	"Continue, you will. Inform the consul of that we shall." The Arach says, bringing out a device and seemingly confirming you'll be heading to <destination>. Without giving you much window to protest, the three thank you for your compliance, and board a Heliarch ship.`
				accept
	on complete

> 		fail "Rim Archaeology 5B"

Author: That's the Zug Wardragon mission. This is so the mission can spawn in the same Wardragon with the scan objective, without there being two of them there at once.

> 		fail "Rim Archaeology 5B"

Reviewer 13: What about after this mission is complete? The Wardragon won't respawn, will it? I feel like it's undesirable to permanently remove the Wardragon without replacing it; it also makes me wonder what happens if the player hasn't already completed `Rim Archaeology 5B`. Will it just not offer?

> 		fail "Rim Archaeology 5B"

Author: It will, `mission "Keep Zug Wardragon"` is in place afterwards with the same behavior as the original mission.

				`	"Don't worry, I'm sure you'll be alright. Good luck on your research."`
					goto end
			`	"True that is, but repealed, the measure that prohibits everyone leaving was not. Spoken with Arbiter Sedlitaris myself I did, and admit I must that much more at ease I now am. But, still, remain vigilant for the crew's sake, I must. Changed their opinions like me, not many of them have."`
			label end
			`	He thanks you again for your help, and with a salute bids you farewell, heading to help the Punisher's crew establish their base of operations inside the caverns. You get back in your ship, and fly to the empty docks of this planet. Now it's a matter of waiting for the Heliarch teams here to do their job and return to the Coalition.`
	on fail
		"reputation: Heliarch" = -1000
		"reputation: Coalition" = -1000

event "heliarch expedition break 2"

mission "Heliarch Expedition 5"
	name "Meet With Aulori"
	description "Head to <destination>, where Consul Aulori wishes to speak with you about the Heliarch's findings on the destroyed ringworld."
	landing
	source
		near "Sol" 1 100
	destination "Ring of Wisdom"
	to offer
		has "event: heliarch expedition break 2"
	on offer
		conversation

> 			`When you've touched down on the landing pads, you see you've received a message from the Heliarchs you brought to Ruin, the mysterious world on the other side of the wormhole the Pug left in Deneb. "Captain <last>, finished what research we could do on the broken ring, we have. About to jump back into the Coalition, we are. Imagine I do that like to speak with you after receiving our report, Consul Aulori will, so head you should to the <planet> as soon as possible."`

Author: After you've brought them there and they're familiar with the concept of how to cross the wormhole, they can make their way back on their own.

			`	He wishes you good luck, and the trio from earlier escorts you back to your ship.`
				accept
	on stopover
		dialog `As instructed, you release the drones as you come in for a landing. Judging by the normal conditions of the spaceport, no one seems to have noticed anything as you pick up the cargo.`
	on complete
		payment 330000
		conversation
			`Landing back on <planet>, you're met by a squad of armed Heliarchs, who are escorting Consul Plortup. He hands you your payment of <payment> with an approving nod.`
			`	"Aided us once more, you have, Captain. To further work with you, I look forward. Search for one of my subordinates in the local spaceport anytime, you should. More work for you soon, I might have." He performs a brief salute, and leaves for some other section of the ring.`

mission "Heliarch Containment 4-A"

> 	name "Undercover"

Author: Changed to "Undercover Transports" and "Meet with Plortub" (altered a bit to have it said that you'll be meeting him in Station Cian, you're not but it just says you will). Better?

	description "Escort three Kimek Spires full of Heliarch agents to <destination>, where they hope the use of civilian ships will give them the element of surprise."
	to offer
		has "event: plasma turret available"
		has "Heliarch Containment 2: done"
		has "Heliarch Containment 3: done"
		not "Resistance: Questions: done"
	source "Ring of Power"
	destination "Remote Blue"
	on offer
		conversation

> 			`You're stopped by one of the many armed Heliarchs in the spaceport, and are asked to follow him into the restricted sections. You comply, and follow through the corridors, until you're brought before Consul Plortup, in a smaller room than last time. Three Kimek Heliarchs are with him.`

Reviewer 7: ```suggestion
			`You're stopped by one of the many armed Heliarchs in the spaceport, and are asked to follow him into the restricted sections. You comply and are led through the corridors until you're brought before Consul Plortup, in a smaller room than last time. Three Kimek Heliarchs are with him.`
```

			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortup and inform him of what happened. Shortly after, <payment> are added to your account, and you're put on a call with the consul.`
			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and is alarmed to learn that they had not only Heliarch equipment, but had also acquired human weapons. He lets out something you can't quite put, resembling a gasp or a roar; you only know that your ears don't want to hear it again.`
			`	"Bring this report to the other consuls, at once I will. Further tasks, of you we may ask, but for now, done your work is, Captain," he ends the transmission after a short salute. Heliarch ships rush back in as the local garrison returns from Remote Blue. Many of the ship captains come to personally congratulate you for stopping the fleeing ships.`

event "lunarium fleeing rb"
	system "12 Autumn Above"
		remove fleet "Heliarch"

event "lunarium detained rb"
	system "12 Autumn Above"
		add fleet "Heliarch" 800

mission "Heliarch Drills 1"

> 	name "Refuelling Drill"

Reviewer 7: ```suggestion
	name "Refueling Drill"
```

> 	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in emergency situations."

Reviewer 7: ```suggestion
	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refueling capabilities in emergency situations."
```

		not "Resistance: Questions: done"
		random < 8
		"combat rating" >= 8104
	on offer
		conversation
			`A pair of Heliarch Judicators land by your ship as you leave for the spaceport. The workers storm to the ships, getting ready to refuel them. Coming down from one of the Judicators, a young Saryd with black hair barely reaching her shoulders pulls out a watch of sorts, seeming to time how long they take to completely fill up the ships' fuel tanks.`
			`	When they're done, she gives an approving nod, and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refuelling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part it is, in testing how well prepared for emergencies we are."`
			choice
				`	"What sort of emergencies?"`
				`	"And you want me to help with some of these drills, I take it?"`
					goto help
				`	"It's good that you do that. I should get going now, though."`
					decline

> 			`	"All sorts of emergencies," she responds. "Potential Quarg invasions, potential criminal attacks, response action to natural disasters, ready for all of them, we need to be."`

Reviewer 7: ```suggestion
			`	"All sorts of emergencies," she responds. "Quarg invasions, criminal attacks, response action to natural disasters, ready for all of them, we need to be."`
```

			label help

> 			`	She takes a look at your ship, then to the two Judicators. "About done with the refuelling drills, we were. Heading back to <planet>, I was. Some more interesting drills, I could set up, if your assistance, I had. Accompany us back there, would you?"`

Reviewer 7: ```suggestion
			`	She takes a look at your ship, then to the two Judicators. "About done with the refueling drills, we were. Heading back to <planet>, I was. Some more interesting drills, I could set up, if your assistance, I had. Accompany us back there, would you?"`
```

			choice
				`	"Alright, what do you need me to do?"`
					goto accept
				`	"What would these 'more interesting drills' involve?"`
				`	"Sorry, I'm not interested in that kind of work."`
					decline
			`	"Thought of that, I still haven't," she says. "Some ship combat training, potentially. Very beneficial, training with an outsider could be."`
			choice
				`	"Alright, so should I just follow your ships now?"`
				`	"Sorry, but I'm not interested in that kind of work."`
					decline
			label accept

> 			`	"Follow you to <planet>, my ships will," she says, pointing at the Judicators. "In a rush to head back, we are not, so if immediate business elsewhere, you have, follow you for a while, we could. An extension to the refuelling drills, it would be."`

Reviewer 7: ```suggestion
			`	"Follow you to <planet>, my ships will," she says, pointing at the Judicators. "In a rush to head back, we are not, so if immediate business elsewhere, you have, follow you for a while, we could. An extension to the refueling drills, it would be."`
```

			`	She tells you she'll pay you <payment> when you reach <planet>, and will prepare another "drill" for you to help with too. She heads back inside one of the Judicators, and they prepare to follow your ship when you launch.`
				accept
	npc accompany save
		personality escort heroic opportunistic
		government "Heliarch"

> 		fleet

Author: When I playtested yes, so provided nothing was broken or messed up during the many commits it should still be working fine.
Probably better to have it confirmed again though.

	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "4 Axis"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Sol Kimek"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "3 Spring Rising"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "5 Summer Above"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "3 Pole"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "12 Autumn Above"
	on visit
		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`

mission "Heliarch Drills 2-C"
	landing
	name "Pursuit Drill"

> 	description "Land on the <planet> while avoiding the Heliarch Hunters."

Reviewer 13: ```suggestion
	description "Land on <planet> while avoiding the Heliarch Hunters."
```

			variant
				"Heliarch Hunter"
		system "Speloog"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Torbab"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Flugbu"
	npc evade
		government "Heliarch"
		personality heroic opportunistic target
		fleet
			names "heliarch"
			variant
				"Heliarch Hunter"
		system "Debrugt"
	on visit
		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`
	on complete
		"coalition jobs" ++
		payment 470000
		conversation
			`Magister Lotuora meets you when you land, handing you your payment of <payment>, and congratulating you for managing to evade the pursuing ships. "Frustrated many of the Hunter pilots, I heard you have," she says with a laugh. "Good training for them, I hope this was. Finishing up the arrangements for one more drill, I am. When ready, you are, meet me in the spaceport, you must."`

mission "Heliarch Drills 3"
	name "Combat Drill"

> 	description "Head to the <waypoints> system to fight the <npc>, reporting back to <planet> when you succeed. You must only disable the ship. Destroying or capturing it would deem you an enemy of the Heliarchs."

Reviewer 13: ```suggestion
	description "Head to the <waypoints> system to fight the <npc>, and report back to <planet> when you succeed. You must only disable the ship. Destroying or capturing it would deem you an enemy of the Heliarchs."
```

	source "Belug's Plunge"
	waypoint "Torbab"
	to offer
		has "Heliarch Drills 2-C: done"
		not "Lunarium: Questions: done"
	on offer
		conversation
			`Lotuora is speaking with a Kimek Heliarch captain at the entrance to the spaceport. "Ah, there the Captain is," she tells the Kimek. "Captain <last>, your sparring partner, this is," she says, gesturing to the Kimek, who salutes you with a short bow of her head. "Excited to test her ship against an outsider, she is."`
			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, where she boards a Punisher and flies away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall."`

> 			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`

Reviewer 13: Can you add a mention of what the payment will be into this conversation? I'll let you decide where. Don't worry about putting it in the description too; that's probably wordy enough already,

				accept
	npc disable save
		government "Heliarch Test Dummy"
		personality disables staying nemesis heroic
		system "Torbab"
		fleet
			names "heliarch"
			variant
				"Heliarch Punisher (Scrappy)"
	on fail
		"reputation: Heliarch" = -1000
		"reputation: Coalition" = -1000
	on complete
		"coalition jobs" ++
		"assisted heliarch" ++
		payment 800000
		conversation
			`Magister Lotuora greets you when you return. "Prevailed over one of our Punishers, you have! Perhaps the real threat, you are, not the Quarg," she says, looking at you for a few seconds as if expecting you to laugh. She hands you <payment>, and continues, "Sent a recording of the fight, I was. Learn much from it, I hope we can."`

> 			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She finally wishes you safe travels, then heads to a Heliarch building.`

Reviewer 13: ```suggestion
			`	She thanks you for helping her with the drills, saying it will be a great help in training the cadets at the local academy. She concludes by wishing you safe travels, and then heads to a Heliarch building.`
```

mission "Heliarch License"
	name "Meet With The Consuls"
	description "Head to the <destination>, where the Heliarch consuls will make you a Heliarch agent."
	minor
	landing
	source
		near "Quaru" 1 100
	destination "Ring of Friendship"

> 	to offer

Reviewer 13: Seems odd that `Heliarch: Investigation` isn't a requirement here, especially since that's the mission blocking participation in the Lunarium intro.

Author: The chains work the same as with the Lunarium Intro, you need any 3 of the 5 to get the join offer, + jobs (or 4 or 5 with less jobs). Investigation blocks Lunarium in general because it's one where you transport Heliarch agents in your ship throughout.

		or
			"assisted heliarch" == 5
			and
				"coalition jobs" >= 90
				"assisted heliarch" == 4
			and
				"coalition jobs" >= 120
				"assisted heliarch" == 3
		not "Lunarium: Questions: done"
	to fail
		has "Lunarium: Questions: done"
	on offer
		conversation
			`When you land on <origin>, you find a Heliarch delegation waiting for you outside. When you meet them, the Saryd of the group says, "A great ally to our Coalition, you have proven to be."`

> 			`	"Discussed your exploits, the consuls have. Much contentment, your contributions brought us," the Kimek continues.`

Reviewer 13: I don't think `contentment` is a good word to use here. Are they more saying you're useful or that they're very happy with you?

Author: Changed to satisfaction, does it work better?

			`	"With you, a meeting they request. To <destination> you must go, if to join our ranks, you so wish," the Arach finishes.`
				accept
	on complete

> 		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology. Was also given a translation device, to freely converse with the Coalition species."

Reviewer 13: ```suggestion
		log "Was given a special circlet by the Heliarchs, which grants access to some of their technology. Was also given a translation device to freely converse with the Coalition species."
```

			`	The consuls who brought you here speak, and once again you're reminded of your "contributions and loyalty" to the Coalition. "Truly special, this ceremony is," the Arach says. "Join our ranks, for the first time, an outsider does. Contributed much and shown devotion to our Coalition, this outsider has."`
			`	"From humanity, <first> <last> has come," the Kimek continues. "Our most recent neighbor, and a group still under the chiding eyes of the Quarg, they are. To reach them all, to free them all, we still cannot, but, our first step in that front, Captain <last>'s coronation is."`
			`	The Saryd approaches you, carrying some type of chest. Opening it, she reveals a golden circlet inside, staying silent and looking at you.`
			choice
				`	"Do I put it on myself?"`
				`	(Take the circlet.)`
					goto circlet
			`	"Allow others to crown us, we do not," she says. "Not anymore."`
			choice
				`	(Take the circlet.)`
			label circlet

> 			`	You pick up the circlet and raise it to your head, the Heliarch consuls all standing up when you do. You bring it down and adjust it so that it fits you well. Once the "crowning" is done, many of the consuls cheer in celebration, with something akin to a round of applause, and more of the speeches from before.`

Reviewer 13: ```suggestion
			`	You pick up the circlet and raise it to your head, the Heliarch consuls all standing up when you do. You bring it down and adjust it so that it fits you well. Once the "crowning" is done, many of the consuls cheer in celebration, which is followed with something akin to a round of applause and more of the speeches from before.`
```

			`	When it's all over, you're escorted out by some of the guards, but remain in the restricted sections. Here you are now brought to a much smaller room, where a pair of Heliarchs in lab coats are working at some device reminiscent of a hospital's ultrasound machine. One of them asks you for your circlet, attaching it to various cables near the machine's controls. "Configure your circlet we now shall, so that to your will it responds," the other says, gesturing for you to take a seat. He holds a large, helmet-like piece, with a cable at the top connecting to the machine.`
			choice
				`	"To my will? What do you mean?"`
				`	"Is this really necessary?"`

> 			`	"Mere jewelry pieces or symbols of rank these are not," the one tinkering with your circlet explains. "Serve they do as links, interfaces between a Heliarch's will and our Coalition's machines and programs. Allow you they will to access some of our restricted ring sections, and serve as well as keys to our many outfitters, they do."`

Reviewer 13: ```suggestion
			`	"Mere jewelry pieces or symbols of rank, these are not," the one tinkering with your circlet explains. "Serve they do as links, interfaces between a Heliarch's will and our Coalition's machines and programs. Allow you they will to access some of our restricted ring sections, and serve as keys to our many outfitters, they do."`
```

> 			`	They gesture for you to sit, and you comply, having the helmet placed over your head. It blocks out your sight completely, and feels a lot heavier that you had imagined. "Begin now, we shall," you barely hear one of the Heliarchs say, as most of the sound is muffled. Once the process starts, you hear some wave-like humming echo inside your head, and it doesn't take long before you feel as if you're being spun around. Before you get too uncomfortable, the Heliarchs stop the machine, and take the helmet off of you. "All done, we are. Here," they say, handing you your circlet back.`

Reviewer 13: ```suggestion
			`	They gesture for you to sit and you comply, having the helmet placed over your head. It blocks out your sight completely, and it feels a lot heavier that you had imagined. "Begin now, we shall," you barely hear one of the Heliarchs say, as most of the sound is muffled. Once the process starts, you hear some wave-like humming echo inside your head, and it doesn't take long before you feel as if you're being spun around. Before you get too uncomfortable, the Heliarchs stop the machine and take the helmet off of you. "All done, we are. Here," one says, handing you your circlet back.`
```

			`	When you're done, the two Heliarchs explain some more. "As of a lower rank you are, permit you access to everything and everywhere, your circlet will not. Not yet, at least. But, as mentioned, come to some of the restricted sections, you now many. On many of our worlds, find you will Heliarch outfitters, where now available some of our equipment will be. Permit you to take on some jobs we put up for Heliarch agents here in the restricted sections, the circlet also will."`
			`	They commend you for having joined up, and bid you farewell, having the guards now bring you back to your ship. The three Heliarch consuls who greeted you are waiting for you when you arrive, and hand you a small box which you soon realize is a translation device. "One of us you now are, Captain," the Saryd says. "Trust you we do to uphold the Coalition's values and laws."`

> 			`	The trio congratulates you once again, and wish you safe travels.`

Reviewer 13: ```suggestion
			`	The trio congratulates you once again and wishes you safe travels.`
```

mission "Heliarch Investigation 2"
	name "Heliarch Investigation"
	description "Take the Heliarch agents to the marked planets so that they can investigate signs of criminal activity."
	passengers 3
	to offer
		has "Heliarch Investigation 1: done"
		not "Lunarium: Questions: done"
	source "Ring of Friendship"
	destination "Ring of Friendship"
	to complete
		has "Heliarch Investigation 2 - Mebla's Portion: done"
		has "Heliarch Investigation 2 - Stronghold of Flugbu: done"
		has "Heliarch Investigation 2 - Shifting Sand: done"
		has "Heliarch Investigation 2 - Fourth Shadow: done"
		has "Heliarch Investigation 2 - Into White: done"
		has "Heliarch Investigation 2 - Remote Blue: done"
	on offer
		conversation
			`Despite the immense traffic in <origin>'s spaceport, the Heliarch trio you brought here is quick to find you after you enter. Kirrie is the first to greet you. "Captain <last>! Glad I am, that still around, you are."`
			`	"Accompany us, would you? A job in mind for you, our superior has." Iplora says.`
			choice

> 				`	"I don't see why not, lead the way."`

Reviewer 11: I would do it like this:
```suggestion
				`	"I don't see why not; lead the way."`
```

> 				`	"I don't see why not, lead the way."`

Reviewer 7: I would have it as two separate sentences. Feels weird to have a semicolon in casual dialog.
```suggestion
				`	"I don't see why not. Lead the way."`
```

				`	"Not this time, I'm really busy now."`
					defer
			`	The three of them escort you through a number of restricted passageways, dotted with armed, statue-like guards at every door and hallway you pass through. Eventually you reach a meeting room of sorts, with about a dozen other Heliarchs seated at the central table. One of them motions for you to join them.`
			`	As you take a seat, one of the Kimek there speaks. "The honorable Captain, we salute. Make you feel welcome, I, Consul Iekie of the Heliarch council, hope we will."`
			choice

> 				`	"Thank you. The honor's all mine."`

Reviewer 14: The player would be saying, 'the honor is all mine?' right? That should be fine.

				accept
	npc "scan outfits"
		personality staying uninterested target
		government "Quarg"
		system "Enif"
		fleet
			names "quarg"
			variant
				"Quarg Skylark"
				"Quarg Wardragon"
	on complete
		"coalition jobs" ++
		payment 200000
		conversation
			`Once again the engineers remove your outfit scanner, take a copy of the scan logs, and hand it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`
			`	When the engineers return, they put your outfit scanner back in place. The Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
			`	He wishes you safe travels and leaves with his colleagues.`

mission "Quarg Interrogation"
	invisible
	landing
	deadline 1
	source
		government "Quarg"

> 		not planet "Humanika"

Author: Changed to attributes quarg, as I think actually it'd be best for this to be as easy to see as possible.
The idea being the Quarg here is one of the Coalition ones, I think it knowing english and being around wherever the player first lands would be okay considering teleport shenanigans.

				goto end
			label hinder
			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
			label end
			`	It blinks a few more times as a few small stripes on its face rapidly alternate between shades of deep blue and bright red. It then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs,' human." It walks away, headed to the Wardragon it looked at before.`
				decline
	npc save
		government "Quarg"
		personality launching uninterested
		fleet
			names "quarg"
			variant
				"Quarg Wardragon"

mission "Heliarch Recon 3-A"
	name "Preparations for Scanning"
	description "Head to <destination>, where the Heliarchs will discuss the next recon task they have for you."
	landing
	source
		near "Ekuarik" 1 100
	destination "Ring of Wisdom"
	to offer
		has "Quarg Interrogation: offered"
		has "Rim Archaeology 5B: active"
		has "event: rim archaeology results"

> 		not "Lunarium: Questions: done"

Author: That mission is the Zug Wardragon mission, meaning for this to offer one must have completed Rim Archaeology (at which point the Wardragon stays over Zug indefinetly), and not have killed the Wardragon.

				has "Heliarch License: done"
			`Three Heliarch officers approach you as you're walking around the spaceport. "Greetings, Captain <last>. Interested, would you be, in accompanying us? A proposition, we have," the Saryd says.`
			choice
				`	"What's it about?"`
				`	"Sure, lead the way."`
					goto way
				`	"Sorry, not up for taking a stroll right now."`
					decline
			`	"Inform you of that, we cannot," the Kimek says. "Not until accepted our offer, you have."`
			choice
				`	"Fine. Bring me to somewhere you can tell me."`
				`	"That seems a bit sketchy, I'll pass."`
					decline
			label license
			`Three Heliarch officers approach you as you're walking around the spaceport. They salute you, and the Saryd speaks. "Greetings, Captain <last>. Sent by Consul Plortub, we were."`
			`	"A mission for you, he has, but to discuss it in private, he wishes to." The Kimek explains. "Follow us, would you?"`
			label way
			`	After a few dozen twists and turns through the restricted sections of the ring, they bring you to a room where an Arach is waiting, alongside a few other Heliarchs who are discussing something while pointing at a map of the Coalition projected over the table.`

> 			`	"Ready to work with us, are you, Captain?" the Arach asks. "The one who asked for you, I am. Call me Consul Plortup, you may."`

Reviewer 15: Is it Consul Plortup or Plortub? See below with Wiz's comment. Make sure you change all references in this mission(s?) to the proper one.

			choice
				`	"<first> <last>. Pleasure to meet you."`
				`	"What job did you have for me?"`
			`	"In need of one such as you, we are. Anonymous reports, we have received, from loyal patriots and colleagues alike. On <stopovers>, a Saryd world, terrible activity may be brewing. Rising in activity there, outlaw groups are, we fear," he says. "Your destination, that is."`

> 			`	A couple small boxes are brought inside and set on the table. Upon opening them, Plortup shows you they contain several small drones - drones very similar to those employed by the repair modules used here.`

Reviewer 15: ```suggestion
			`	A couple small boxes are brought inside and set on the table. Upon opening them, Plortup shows you they contain several small drones - drones very similar to those employed by the repair modules used here.`
```
Where is here? What for? Although I know what it's referencing, this seems a little out of left field.

Author: Changed to `...very similar to those employed by the hull repair modules used in Coalition ships.`

		ship "Kimek Spire" "Kiry Paichee"
		ship "Kimek Spire" "Achipa Kyrri"
		ship "Kimek Spire" "Pakee Pikary"
	on visit
		dialog `You've reached <planet>, but the Heliarch transports haven't arrived yet! Best depart and wait for them to get here.`

mission "Heliarch Containment 4-B"
	name "Meet with Plortub"
	description "Collect your payment for escorting the Heliarch transports."
	landing
	to offer
		has "Heliarch Containment 4-A: done"
		not "Lunarium: Questions: done"
	source "Remote Blue"
	destination "Station Cian"
	on offer
		conversation
			`Hundreds of Heliarch agents pour out of the three Kimek transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand. From the reactions of the locals, it's clear they were neither expecting such a thing, nor are they happy with the agents going about the city in that manner.`
			`	When the last of the Heliarchs leave the transports, one of the Kimek captains inform you that you should now head to <destination> to meet with Consul Plortub collect your payment of <payment>, and to "avoid getting caught in any potential retaliation."`
				accept
	on complete
		payment 495000
		conversation

> 			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`

Reviewer 13: ```suggestion
			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortub. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`
```

	description "Defeat the criminal ships that are attempting to flee. You must disable and board their ships in order to help the Heliarchs find the source of their weapons."
	landing
	to offer
		has "Heliarch Containment 4-B: done"
		not "Lunarium: Questions: done"
	source "Station Cian"
	on offer
		"reputation: Lunarium" = -1000
		event "lunarium fleeing rb"
		conversation
			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything particularly eye-catching, as the station holds few shops. You move to the job board, but find it completely empty. You also notice that the market's trade values are not updating, and any news system is non-functional.`
			`	Suddenly, some of the few Heliarchs still stationed here rush past you heading for an office. They quickly reemerge and approach you.`
			`	"Captain! Helped us, you have, yes? Aiding in the Remote Blue operation?" You nod. "Think you trustworthy, the consul does, and in dire need of a combat-ready ship, we are. Aid us, you must!"`
			choice
				`	"What's going on?"`

> 				`	"My ship isn't exactly combat-ready to fight your own warships."`

Reviewer 15: ```suggestion
				`	"My ship isn't exactly combat-ready to fight your own warships."`
```
It feels a little odd to have you say this, but then also be forced to fight anyways. Maybe there should be some sort of decline option in this mission.

Author: Changed to `"What do you mean combat-ready? Are we being attacked?"`
I don't think this mission should be declineable at this point, it is the last mission of the chain and would frankly be inappropriate as I see it if you can simply dip on the Heliarchs here when you've been pretty willing. I don't thinki "forcing" at this point is really "forcing".

	minor
	source
		near "Belug" 6 100
	destination "Belug's Plunge"
	to offer
		has "Coalition: Contributor: done"
		not "Lunarium: Questions: done"
		random < 8
		"combat rating" >= 8104
	on offer
		conversation
			`A pair of Heliarch Judicators land by your ship as you leave for the spaceport. The workers storm to the ships, getting ready to refuel them. Coming down from one of the Judicators, a young Saryd with black hair barely reaching her shoulders pulls out a watch of sorts, seeming to time how long they take to completely fill up the ships' fuel tanks.`
			`	When they're done, she gives an approving nod and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refueling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part it is, in testing how well prepared for emergencies we are."`
			choice
				`	"What sort of emergencies?"`

> 				`	"And you want me to help with some of these drills, I take it?"`

Reviewer 7: I think it's fine.

			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, where she boards a Punisher and flies away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall. If successful, you are, set aside <payment> for your efforts, we have."`
			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`
				accept
	npc disable save
		government "Heliarch Test Dummy"
		personality disables staying nemesis heroic
		system "Torbab"
		fleet
			names "heliarch"
			variant

> 				"Heliarch Punisher (Scrappy)"

Author: I fully intend on having the Heliarch side start off easier but climb to much harder in terms of combat difficulty, as you go from fighting much weaker civilian ships to Quarg HWs. So, while upon beginning it will not warrant much, by the end you should be very well armed.
This mission set checks for CR >= 8104 before starting up at all; I could increase that to some other value if it's preferred.
The final mission of this chain is one without time constraints, with a single ship that's stuck in an uninhabited system that one can easily avoid jumping to. I don't see why the ship should be toned down considering that the duel with it is pretty much all the challenge for this mission set.
Will edit the convo to add her saying you can just go and tackle the ship whenever you feel you're ready, as there's no rush.

			choice
				`	"I'm not sure I understand what you're getting at."`
				`	"Obviously I have my limits as for what I would do."`

> 			`	"A Jump Drive, you possess. Far away, you can travel. Bother you, the harsh cages of the Quarg do not," the Saryd explains as all three walk toward your ship.`

Reviewer 4: Jump drives are one of the few outfits that should remain lowercase.

		payment 200000
		conversation
			`Once again the engineers remove your outfit scanner, taking a copy of the scan logs and handing it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`

> 			`	When the engineers return, they put your Outfit Scanner back in place. The Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`

Reviewer 4: So are outfit scanners.

			`	He squints his eyes, then speaks, "Repeat that, could you?" You say it again, and he once more looks puzzled. He gestures to a Heliarch beside him, then takes off the translation box from around his neck, swapping it for theirs.`
			`	He asks again, and you repeat it again. "An... incomplete ringworld, is it?"`
			choice

> 				`	"No, it is definetly broken. There's tons of debris littering the system, and no sign of life in it."`

Reviewer 7: ```suggestion
				`	"No, it is definitely broken. There's tons of debris littering the system, and no sign of life in it."`
```
